---
weight: 10
bookFlatSection: false
title: "Chapter 10"
description: "Perturbations to metabolic networks"
---

# Chapter 10: Perturbations to metabolic networks

{{< hint info >}}
**Fast-SL: an efficient algorithm to identify synthetic lethal sets in metabolic networks**   
[Pratapa _et al._  (2015)](http://doi.org/)

Describes a novel and rapid method for identifying synthetic lethals in metabolic networks, by cutting through the search space.
{{< /hint >}}
{{< hint info >}}
**Genome-scale gene/reaction essentiality and synthetic lethality analysis**   
[Suthers _et al._  (2009)](http://doi.org/)

Describes SL-Finder, an MILP-based method to identify synthetic lethals.
{{< /hint >}}
{{< hint info >}}
**Enumeration of Smallest Intervention Strategies in Genome-Scale Metabolic Networks**   
[von Kamp and Klamt (2014)](http://doi.org/)

Describes MCSEnumerator, an efficient MILP-based method to identify synthetic lethals.
{{< /hint >}}
{{< hint info >}}
**Global reconstruction of the human metabolic network based on genomic and bibliomic data**   
[Duarte _et al._ (2007)](http://doi.org/)

The classic reconstruction of human metabolic network, ""Recon 1
{{< /hint >}}
{{< hint info >}}
**A community-driven global reconstruction of human metabolism**   
[Thiele _et al._ (2013)](http://doi.org/)

The Recon 2 human metabolic model.
{{< /hint >}}
{{< hint info >}}
**Computational reconstruction of tissue-specific metabolic models: application to human liver metabolism**   
[Jerby _et al._ (2010)](http://doi.org/)

Tissue-specific metabolic reconstruction.
{{< /hint >}}
{{< hint info >}}
**Insight into human alveolar macrophage and _M. tuberculosis_ interactions via metabolic reconstructions**   
[Bordbar _et al._ (2010)](http://doi.org/)

Modelling host&ndash;pathogen interactions using metabolic modelling.
{{< /hint >}}
{{< hint info >}}
**A multi-tissue type genome-scale metabolic network for analysis of whole-body systems physiology**   
[Bordbar _et al._ (2011)](http://doi.org/)

Tissue-specific metabolic reconstruction, for Recon 1, modelling adipocytes, hepatocytes, and myocytes.
{{< /hint >}}
{{< hint info >}}
**Reconstruction of Genome-Scale Active Metabolic Networks for 69 Human Cell Types and 16 Cancer Types Using INIT**   
[Agren _et al._ (2012)](http://doi.org/)

Excellent example of context-specific metabolic reconstructions.
{{< /hint >}}
{{< hint info >}}
**Predicting selective drug targets in cancer through metabolic networks**   
[Folger _et al._ (2011)](http://doi.org/)

The first genome-scale metabolic model of cancer metabolism.
{{< /hint >}}
{{< hint info >}}
**Large-scale _in silico_ modeling of metabolic interactions between cell types in the human brain**   
[Lewis _et al._ (2010)](http://doi.org/)

A workflow that integrates gene expression data, proteomics data and literature-based manual curation to model human metabolism within and between different types of cells.
{{< /hint >}}
{{< hint info >}}
**A compendium of inborn errors of metabolism mapped onto the human metabolic network**   
[Sahoo _et al._ (2012)](http://doi.org/)

A comprehensive knowledgebase of inborn errors of metabolism.
{{< /hint >}}
{{< hint info >}}
**Predicting the impact of diet and enzymopathies on human small intestinal epithelial cells**   
[Sahoo and Thiele_ (2013)](http://doi.org/)

This paper defined fifty metabolic tasks to evaluate model functionalities.
{{< /hint >}}
{{< hint info >}}
**How Will Bioinformatics Influence Metabolic Engineering?**   
[Edwards and Palsson (1997)](http://doi.org/)

A classic review, discussing computational approaches for metabolic engineering.
{{< /hint >}}
{{< hint info >}}
**Basic concepts and principles of stoichiometric modeling of metabolic networks**   
[Maarleveld _et al._ (2013)](http://doi.org/)

An excellent review covering the breadth of metabolic modelling.
{{< /hint >}}
{{< hint info >}}
**The Convex Basis of the Left Null Space of the Stoichiometric Matrix Leads to the Definition of Metabolically Meaningful Pools**   
[Famili and Palsson (2003)](http://doi.org/)

A nice paper providing detailed insights into stoichiometric matrix and the convex basis of the metabolic solution spaces.
{{< /hint >}}
{{< hint info >}}
**Systemic metabolic reactions are obtained by singular value decomposition of genome-scale stoichiometric matrices**   
[Famili and Palsson (2003)](http://doi.org/)

SVD analysis of the stoichiometric matrix.
{{< /hint >}}
{{< hint info >}}
**Constraining the metabolic genotype&ndash;phenotype relationship using a phylogeny of _in silico_ methods**   
[Lewis _et al._ (2012)](http://doi.org/)

A detailed review of the entire family of COBRA methods (with a beautiful method phylogeny picture).
{{< /hint >}}
{{< hint info >}}
**Regulation of Gene Expression in Flux Balance Models of Metabolism**   
[Covert _et al._ (2001)](http://doi.org/)

The classic paper on rFBA.
{{< /hint >}}
{{< hint info >}}
**Interpreting Expression Data with Metabolic Flux Models: Predicting _Mycobacterium tuberculosis_ Mycolic Acid Production**   
[Colijn _et al._ (2009)](http://doi.org/)

Describes the E-Flux method for integration of regulatory information.
{{< /hint >}}
{{< hint info >}}
**Probabilistic integrative modeling of genome-scale metabolic and regulatory networks in _Escherichia coli_ and _Mycobacterium tuberculosis_**   
[Chandrasekharan and Price (2010)](http://doi.org/)

Describes the PROM approach for integration of regulatory information from gene expression datasets.
{{< /hint >}}
{{< hint info >}}
**Constraint-based models predict metabolic and associated cellular functions**   
[Bordbar _et al._ (2014)](http://doi.org/)

An excellent review on constraint-based models and their applications.
{{< /hint >}}
{{< hint info >}}
**Identifying gene targets for the metabolic engineering of lycopene biosynthesis in _Escherichia coli_**   
[Alper _et al._ (2005)](http://doi.org/)

A classic application of FBA for metabolic engineering, including identification of a triple knock-out for improving lycopene production.
{{< /hint >}}
{{< hint info >}}
**Flux Balance Analysis of Mycolic Acid Pathway: Targets for Anti-Tubercular Drugs**   
[Raman _et al._ (2005)](http://doi.org/)

A classic application of FBA for drug target identification by modelling the mycolic acid pathway in _Mycobacterium tuberculosis_.
{{< /hint >}}
{{< hint info >}}
**targetTB: A target identification pipeline for _Mycobacterium tuberculosis_ through an interactome, reactome and genome-scale structural analysis**   
[Raman _et al._ (2008)](http://doi.org/)

A rigorous pipeline for drug target identification in _Mycobacterium tuberculosis_.
{{< /hint >}}
{{< hint info >}}
**Applications of genome-scale metabolic reconstructions**   
[Oberhardt _et al._ (2009)](http://doi.org/)

A detailed review, discussing various classes of applications of constraint-based modelling.
{{< /hint >}}
{{< hint info >}}
**The application of flux balance analysis in systems biology**   
[Gianchandani _et al._ (2010)](http://doi.org/)

A nice and simple overview of FBA and related methods, with applications.
{{< /hint >}}
{{< hint info >}}
**Genome-scale metabolic networks**   
[Terzer _et al._ (2009)](http://doi.org/)

A very nice review on various aspects of metabolic network reconstruction and simulation.
{{< /hint >}}
{{< hint info >}}
**Elementary flux modes in a nutshell: Properties, calculation and applications**   
[Zanghellini _et al._ (2013)](http://doi.org/)

A neat introduction to EFM analysis, using a ""metro map"" analogy.
{{< /hint >}}
{{< hint info >}}
**Detection of elementary flux modes in biochemical networks: a promising tool for pathway analysis and metabolic engineering**   
[Schuster _et al._ (1999)](http://doi.org/)

An excellent paper on EFMs.
{{< /hint >}}
{{< hint info >}}
**Elementary mode analysis: a useful metabolic pathway analysis tool for characterizing cellular metabolism**   
[Trinh _et al._ (2009)](http://doi.org/)

A nice review detailing EFMs, especially their applications for metabolic engineering.
{{< /hint >}}
{{< hint info >}}
**Comparison of network-based pathway analysis methods**   
[Papin _et al._ (2004)](http://doi.org/)

A neat illustration of EFM vs. EPA analyses.
{{< /hint >}}
{{< hint info >}}
**On Elementary Flux Modes in Biochemical Reaction Systems at Steady State**   
[Schuster and Hilgetag (1994)](http://doi.org/)

The classic paper on EFMs.
{{< /hint >}}
{{< hint info >}}
**Extreme Pathway Analysis of Human Red Blood Cell Metabolism**   
[Wiback and Palsson (2002)](http://doi.org/)

Discusses EPA, for studying human RBC metabolism.
{{< /hint >}}
{{< hint info >}}
**High-throughput metabolic flux analysis based on gas chromatographyï¿½mass spectrometry derived 13C constraints**   
[Fischer _et al._ (2004)](http://doi.org/)

A classic paper giving a clear rundown of 13C-MFA.
{{< /hint >}}
{{< hint info >}}
**13C Metabolic Flux Analysis**   
[Wiechert (2001)](http://doi.org/)

A classic review on 13C-MFA.
{{< /hint >}}
{{< hint info >}}
**13C-based metabolic flux analysis**   
[Zamboni _et al._ (2009)](http://doi.org/)

A detailed protocol paper elucidating the 13C-MFA approach.
{{< /hint >}}
{{< hint info >}}
**How to measure metabolic fluxes: a taxonomic guide for 13C fluxomics**   
[Niedenfuehr _et al._ (2015)](http://doi.org/)

Another excellent review on 13C-MFA.
{{< /hint >}}
{{< hint info >}}
**13C Metabolic Flux Analysis for Systematic Metabolic Engineering of _S. cerevisiae_ for Overproduction of Fatty Acids**   
[Ghosh _et al._ (2016)](http://doi.org/)

A nice illustration of flux analysis for improving production of fatty acids in yeast.
{{< /hint >}}
