'use strict';(function(){const b={cache:!0};b.doc={id:'id',field:['title','content'],store:['title','href','section']};const a=FlexSearch.create('balance',b);window.bookSearchIndex=a,a.add({id:0,href:'/SysBioBook/resources/appendices/',title:"Appendices",section:"Resources",content:"Appendices #  It is desired to make the book as self-contained as possible; therefore, additional material in the form of five online appendices are also included.\nA. Introduction to Key Biological Concepts #  This appendix provides a quick introduction to biology, focussing on basic biological macromolecules such as nucleic acids, proteins, carbohydrates, lipids, and small molecules. It also provides a brief overview of information flow in biological systems, discussing aspects of DNA replication, mRNA transcription, protein translation, and post-translational modifications.\nB. Reconstruction of Biological Networks #  This appendix provides an overview of reconstruction of various types of biological networks, viz. gene regulatory networks, protein\u0026ndash;protein interaction networks, signalling networks, and metabolic networks.\nC. Databases for Systems Biology #  This appendix presents a brief overview of databases of PPIs and functional associations, metabolic network databases, gene regulatory network databases, model databases, and certain miscellaneous databases.\nD. Software Tools Compendium #  Beginning with a summary of standards and formats, this appendix details various software tools that are useful for all the modelling paradigms studied in the book, such as network biology, dynamic modelling and parameter estimation, Boolean networks, constraint-based modelling, and community modelling.\nE. MATLAB for Systems Biology #  This appendix provides a solid introduction to the basics of using MATLAB, followed by a discussion on how a variety of tools and toolboxes can be used for handling various modelling and simulation tasks in systems biology.\nAppendices are located online-only at https://www.routledge.com/9781138597327.\n"}),a.add({id:1,href:'/SysBioBook/resources/further-reading/chapter-01/',title:"Chapter 1",section:"Further Reading",content:"Chapter 1: Introduction to modelling #  Can a biologist fix a radio?—Or, what I learned while studying apoptosis\nLazebnik Y (2002) Cancer Cell\nThis insightful paper by Yuri Lazebnik discusses some very interesting points related to engineering vs. biology, and underlines the challenges associated with studying biological complexity.\n From molecular to modular cell biology\nHartwell (1999)\nA nice essay on modularity in biology, and a systems perspective, from Leland Hartwell, who went on to win the 2001 Nobel Prize in Physiology or Medicine.\n Modelling viral and immune system dynamics\nPerelson (2002)\nAn authoritative review on modelling the dynamics of viral infections e.g. HIV/HCV.\n Numb3rs 1x03 - Vector (TV episode)\nThe protagonist in this show is a mathematician, who uses SIR modelling to help the FBI track down the spread of a virus in a city.\n "}),a.add({id:2,href:'/SysBioBook/resources/further-reading/chapter-02/',title:"Chapter 2",section:"Further Reading",content:"Chapter 2: Introduction to graph theory #  VAX: Gamifying Epidemic Prevention\nAn engaging game, that uses network theory to model the spread of an epidemic. Players need to stop epidemic spread by vaccinating/quarantining the right \u0026lsquo;nodes\u0026rsquo; (people) from the network.\n Leonhard Euler and the Koenigsberg Bridges\n(1953)\nA nice article recapitulating the history of the Koenigsberg bridges\n The truth about Koenigsberg\nHopkins (2018)\nAnother article on the history of the bridges\n The SuiteSparse Matrix Collection\nThe SuiteSparse Matrix Collection (formerly the University of Florida Sparse Matrix Collection) is a widely used set of sparse matrix benchmarks collected from a wide range of applications.\n MATLAB Graph Functions Reference\nMATLAB has a suite of functions for studying/analysing networks. These are described here.\n "}),a.add({id:3,href:'/SysBioBook/resources/codes/',title:"Codes",section:"Resources",content:"Codes #  The companion code repository to this book includes some key snippets of code that will enable readers to understand some of the models/methodologies described in this book.\nhttps://github.com/RamanLab/SysBioBook-Codes\nThe idea is to expand this repository over a period of time. If you think there is a piece of code that will help you better understand a concept in the book, please contribute an issue at https://github.com/RamanLab/SysBioBook-Codes/issues and I will be more than happy to include one or more code snippets to address it.\n"}),a.add({id:4,href:'/SysBioBook/resources/further-reading/chapter-03/',title:"Chapter 3",section:"Further Reading",content:"Chapter 3: Structure of networks #  Network Biology\nBarabasi and Oltvai (2004)\nA classic review on network biology capturing a very good breadth of topics, ranging from network parameters all the way up to network perturbations.\n How Ideas Spread?\nA very nice article by Duncan Watts, the proponent of the small-world network hypothesis.\n Using graph theory to analyze biological networks\nPavlopoulos et al. (2011)\nA nice review on graph theory applications in network biology.\n Star Wars Social Network! (by Evelina Gabasova)\nA fun exercise, studying the social networks in Star Wars movies!\n Collective dynamics of ‘small-world’ networks\nWatts and Strogatz (1998)\nThe original paper on small world networks\n Complex networks: Structure and dynamics\nBoccaletti et al. (2006)\nExtremely detailed paper on networks (a mini book/reference!)\n Complex Networks: Small-World, Scale-Free and Beyond\nWang and Chen (2003)\nA lucid easy-to-read paper discussing network models\n How Network Theory Is Revealing Previously Unknown Patterns in Sports\nAn interesting read on the use of network science in sports.\n Emergence of Scaling in Random Networks\nBarabasi and Albert (1999)\nThe original paper on scale-free networks.\n Structure and Function of Complex Networks\nNewman (2003)  Network motifs: Simple building blocks of complex networks\nMilo et al. (2002)  FANMOD: a tool for fast network motif detection\nAn easy-to-use software tool for motif detection.\n Aggregation of topological motifs in the Escherichia coli transcriptional regulatory network\nDobrin et al. (2004)  Motifs, themes and thematic maps of an integrated Saccharomyces cerevisiae interaction network\nZhang et al. (2005)  Network motifs in the transcriptional regulation network of Escherichia coli\nShen-Orr et al. (2002)  Error and attack tolerance of complex networks\nBarabasi and Albert (2000)  Attack vulnerability of complex networks\nHolme et al. (2002)  "}),a.add({id:5,href:'/SysBioBook/resources/lecture-videos/',title:"Lecture Videos",section:"Resources",content:"Lecture Videos #  A Massive Open Online Course on Computational Systems Biology, which is in part, a basis for this book, is also available, through the Indian educational platforms, NPTEL (National Programme on Technology Enhanced Learning) and SWAYAM.\nThe lectures (including hands-on lab sessions), totalling over 30 hours of content, are also available on YouTube.\n"}),a.add({id:6,href:'/SysBioBook/resources/further-reading/chapter-04/',title:"Chapter 4",section:"Further Reading",content:"Chapter 4: Applications of network biology #  Lethality and centrality in protein networks\nJeong et al. (2001)  Nine quick tips for analyzing network data\nMiele et al. (2020)  The human disease network\nGoh et al. (2007)  Edgetic perturbation models of human inherited disorders\nZhong et al. (2009)  Differential network biology\nIdeker and Krogan (2012)  Network-based features enable prediction of essential genes across diverse organisms\nAzhagesan et al. (2018)  Adapting Community Detection Algorithms for Disease Module Identification in Heterogeneous Biological Networks\nTripathi et al. (2019)  NetGenes Database  Assessment of network module identification across complex diseases\nChoobdar et al. (2019)  Predicting novel metabolic pathways through subgraph mining\nSankar_et al._ (2017)  Protein structure: insights from graph theory\nVishveshwara et al. (2002)  The Protein Folding Network\nRao and Caflisch (2004)  A General Mechanism for the Propagation of Mutational Effects in Proteins\nRajasekaran et al. (2017)  Toward a quantitative description of microscopic pathway heterogeneity in protein folding\nGopi et al. (2017)  Missing and spurious interactions and the reconstruction of complex networks\nGuimera and Sales-Pardo (2009)  Network representations and methods for the analysis of chemical and biochemical pathways\nSandefur et al. (2013)  "}),a.add({id:7,href:'/SysBioBook/resources/further-reading/',title:"Further Reading",section:"Resources",content:"Further Reading #  While the book contains over 700 references including special pointers to material for further reading, the following annotated reading list separated chapter-wise will be helpful in digging deeper into the concepts discussed in the text.\n  Chapter 1   Introduction to modelling   Chapter 2   Introduction to graph theory   Chapter 3   Structure of networks   Chapter 4   Applications of network biology   Chapter 5   Introduction to dynamic modelling   Chapter 6   Parameter estimation   Chapter 7   Discrete dynamic models: Boolean networks   Chapter 8   Introduction to constraint-based modelling   Chapter 9   Extending constraint-based approaches   Chapter 10   Perturbations to metabolic networks   Chapter 11   Modelling cellular interactions   Chapter 12   Designing biological circuits   Chapter 13   Robustness and evolvability of biological systems   Chapter 14   Epilogue: The road ahead   Appendix A   Introduction to key biological concepts   Appendix B   Reconstruction of biological networks   Appendix C   Databases for systems biology   Appendix D   Software tools compendium   Appendix E   MATLAB for systems biology   "}),a.add({id:8,href:'/SysBioBook/resources/further-reading/chapter-05/',title:"Chapter 5",section:"Further Reading",content:"Chapter 5: Introduction to dynamic modelling #  An Introduction to Dynamical Systems\nSobie (2011)  Quantitative Modeling in Cell Biology: What Is It Good for?\nMogilner et al. (2006)  Biochemical Systems Theory: A Review\nVoit (2013)  "}),a.add({id:9,href:'/SysBioBook/resources/further-reading/chapter-06/',title:"Chapter 6",section:"Further Reading",content:"Chapter 6: Parameter estimation #  Linking data to models: data regression\nJaqaman and Danuser (2006)  Good practices for building dynamical models in systems biology\nAzeloglu and Iyengar (2015)  Reverse engineering and identification in systems biology: strategies, perspectives and challenges\nVillaverde and Banga (2014)  Recent developments in parameter estimation and structure identification of biochemical and genomic systems\nChun-Chou and Voit (2009)  Parameter Estimation in Biochemical Pathways: A Comparison of Global Optimization Methods\nMoles et al. (2003)  Programming biological models in Python using PySB\nLopez et al. (2013)  Dynamical modeling and multi-experiment fitting with PottersWheel\n(2008)  Evolutionary Computation\nFoster (2001)  Sensitivity analysis approaches applied to systems biology models\nZi (2011)  An Introduction to Genetic Algorithms\nMelanie Mitchell (1995)  GAs for Travelling Salesperson Problem  "}),a.add({id:10,href:'/SysBioBook/resources/further-reading/chapter-07/',title:"Chapter 7",section:"Further Reading",content:"Chapter 7: Discrete dynamic models: Boolean networks #  Modelling and analysis of gene regulatory networks\nKarlebach and Shamir (2008)  Boolean network simulations for life scientists\nAlbert et al. (2008)  Boolean Modeling of Genetic Regulatory Networks\nAlbert et al. (2004)  Robustness and fragility of Boolean models for genetic regulatory networks\nChaves et al. (2005)  Algebraic Models of Biochemical Networks\nLaubenbacher and Jarrah (2009)  Dynamical analysis of a generic Boolean model for the control of the mammalian cell cycle\nFaure et al. (2006)  BooleSim  BooleanNet  Predicting Essential Components of Signal Transduction Networks: A Dynamic Model of Guard Cell Abscisic Acid Signaling\nLi et al. (2006)  "}),a.add({id:11,href:'/SysBioBook/resources/further-reading/chapter-08/',title:"Chapter 8",section:"Further Reading",content:"Chapter 8: Introduction to constraint-based modelling #  A protocol for generating a high-quality genome-scale metabolic reconstruction\nThiele and Palsson (2010)  What is flux balance analysis?\nOrth et al. (2010)  Advances in flux balance analysis\nKauffman et al. (2003)  Flux balance analysis of biological systems: applications and challenges\nRaman and Chandra (2009)  "}),a.add({id:12,href:'/SysBioBook/resources/further-reading/chapter-09/',title:"Chapter 9",section:"Further Reading",content:"Chapter 9: Extending constraint-based approaches #  Analysis of optimality in natural and perturbed metabolic networks\nSegre et al. (2002)  Using Genome-scale Models to Predict Biological Capabilities\nO' Brien et al. (2015)  Regulatory on-off minimization of metabolic flux changes after genetic perturbations\nShlomi et al. (2005)  In Silico Identification of Gene Amplification Targets for Improvement of Lycopene Production\nChoi et al. (2010)  Computational identification of gene over-expression targets for metabolic engineering of taxadiene production\nBoghigian et al. (2012)  Predicting Gene Essentiality Using Genome-Scale in silico Models\nJoyce and Palsson (2008)  "}),a.add({id:13,href:'/SysBioBook/resources/further-reading/chapter-10/',title:"Chapter 10",section:"Further Reading",content:"Chapter 10: Perturbations to metabolic networks #  Fast-SL: an efficient algorithm to identify synthetic lethal sets in metabolic networks\nPratapa et al. (2015)  Genome-scale gene/reaction essentiality and synthetic lethality analysis\nSuthers et al. (2009)  Enumeration of Smallest Intervention Strategies in Genome-Scale Metabolic Networks\nvon Kamp and Klamt (2014)  Global reconstruction of the human metabolic network based on genomic and bibliomic data\nDuarte et al. (2007)  A community-driven global reconstruction of human metabolism\nThiele et al. (2013)  Computational reconstruction of tissue-specific metabolic models: application to human liver metabolism\nJerby et al. (2010)  Insight into human alveolar macrophage and M. tuberculosis interactions via metabolic reconstructions\nBordbar et al. (2010)  A multi-tissue type genome-scale metabolic network for analysis of whole-body systems physiology\nBordbar et al. (2011)  Reconstruction of Genome-Scale Active Metabolic Networks for 69 Human Cell Types and 16 Cancer Types Using INIT\nAgren et al. (2012)  Predicting selective drug targets in cancer through metabolic networks\nFolger et al. (2011)  Large-scale in silico modeling of metabolic interactions between cell types in the human brain\nLewis et al. (2010)  A compendium of inborn errors of metabolism mapped onto the human metabolic network\nSahoo et al. (2012)  Predicting the impact of diet and enzymopathies on human small intestinal epithelial cells\nSahoo and Thiele_ (2013)  How Will Bioinformatics Influence Metabolic Engineering?\nEdwards and Palsson (1997)  Basic concepts and principles of stoichiometric modeling of metabolic networks\nMaarleveld et al. (2013)  The Convex Basis of the Left Null Space of the Stoichiometric Matrix Leads to the Definition of Metabolically Meaningful Pools\nFamili and Palsson (2003)  Systemic metabolic reactions are obtained by singular value decomposition of genome-scale stoichiometric matrices\nFamili and Palsson (2003)  Constraining the metabolic genotype–phenotype relationship using a phylogeny of in silico methods\nLewis et al. (2012)  Regulation of Gene Expression in Flux Balance Models of Metabolism\nCovert et al. (2001)  Interpreting Expression Data with Metabolic Flux Models: Predicting Mycobacterium tuberculosis Mycolic Acid Production\nColijn et al. (2009)  Probabilistic integrative modeling of genome-scale metabolic and regulatory networks in Escherichia coli and Mycobacterium tuberculosis\nChandrasekharan and Price (2010)  Constraint-based models predict metabolic and associated cellular functions\nBordbar et al. (2014)  Identifying gene targets for the metabolic engineering of lycopene biosynthesis in Escherichia coli\nAlper et al. (2005)  Flux Balance Analysis of Mycolic Acid Pathway: Targets for Anti-Tubercular Drugs\nRaman et al. (2005)  targetTB: A target identification pipeline for Mycobacterium tuberculosis through an interactome, reactome and genome-scale structural analysis\nRaman et al. (2008)  Applications of genome-scale metabolic reconstructions\nOberhardt et al. (2009)  The application of flux balance analysis in systems biology\nGianchandani et al. (2010)  Genome-scale metabolic networks\nTerzer et al. (2009)  Elementary flux modes in a nutshell: Properties, calculation and applications\nZanghellini et al. (2013)  Detection of elementary flux modes in biochemical networks: a promising tool for pathway analysis and metabolic engineering\nSchuster et al. (1999)  Elementary mode analysis: a useful metabolic pathway analysis tool for characterizing cellular metabolism\nTrinh et al. (2009)  Comparison of network-based pathway analysis methods\nPapin et al. (2004)  On Elementary Flux Modes in Biochemical Reaction Systems at Steady State\nSchuster and Hilgetag (1994)  Extreme Pathway Analysis of Human Red Blood Cell Metabolism\nWiback and Palsson (2002)  High-throughput metabolic flux analysis based on gas chromatography�mass spectrometry derived 13C constraints\nFischer et al. (2004)  13C Metabolic Flux Analysis\nWiechert (2001)  13C-based metabolic flux analysis\nZamboni et al. (2009)  How to measure metabolic fluxes: a taxonomic guide for 13C fluxomics\nNiedenfuehr et al. (2015)  13C Metabolic Flux Analysis for Systematic Metabolic Engineering of S. cerevisiae for Overproduction of Fatty Acids\nGhosh et al. (2016)  "}),a.add({id:14,href:'/SysBioBook/resources/further-reading/chapter-11/',title:"Chapter 11",section:"Further Reading",content:"Chapter 11: Modelling cellular interactions #  Host�pathogen systems biology\nForst (2006)  Modelling Viral and Immune System Dynamics\nPerelson (2002)  Immunology for physicists\nPerelson (1997)  Boolean models of within-host immune interactions\nThakar and Albert (2010)  Constraint-based analysis of metabolic capacity of Salmonella typhimurium during host-pathogen interaction\nRaghunathan et al. (2009)  A systems perspective of host�pathogen interactions: predicting disease outcome in tuberculosis\nRaman et al. (2010)  Current trends in modeling host�pathogen interactions\nMukherjee et al. (2013)  "}),a.add({id:15,href:'/SysBioBook/resources/further-reading/chapter-12/',title:"Chapter 12",section:"Further Reading",content:"Chapter 12: Designing biological circuits #  Introduction to Synthetic Biology\n   Synthetic Biology, Inventing the Future by Andrew Hessel\n   Synthetic Biology: Honors Colloquium by JJ Collins\n   Realizing the potential of synthetic biology\nChurch et al. (interview) (2014)  Foundations for engineering biology\nEndy (2005)  Genome-scale engineering for systems and synthetic biology\nEsvelt and Wang (2013)  The second wave of synthetic biology: from modules to systems\nPurnick and Weiss (2009)  "}),a.add({id:16,href:'/SysBioBook/resources/further-reading/chapter-13/',title:"Chapter 13",section:"Further Reading",content:"Chapter 13: Robustness and evolvability of biological systems #  Biological Robustness\nKitano (2004)  Bow ties, metabolism and disease\nCsete and Doyle (2004)  Complexity and Robustness\nCarlson and Doyle (2002)  Evolution of Bow-Tie Architectures in Biology\nFriedlander et al. (2005)  Neutralism and selectionism: a network-based reconciliation\nWagner (2008)  Innovation and robustness in complex regulatory gene networks\nCiliberti et al. (2007)  Robustness and evolvability: a paradox resolved\nWagner (2008)  Andreas Wagner interview (Zürich Minds)\n   "}),a.add({id:17,href:'/SysBioBook/resources/further-reading/chapter-14/',title:"Chapter 14",section:"Further Reading",content:"Chapter 14: Epilogue: The road ahead #  The past 200 years in diabetes\nPolonsky (2012)  Physiological regulation of the pancreatic β-cell:functional insights for understanding and therapy of diabetes\nMcClenaghan (2007)  Insulin: discovery and controversy\nRosenfeld (2002)  The impact of mathematical modeling on the understanding of diabetes and related complications\nAjmera et al. (2013)  Entelos PhysioLab Models  Bridging the gaps in systems biology\nCvijovic et al. (2014)  Simulating a Living Cell\nCovert (2014)  A Whole-Cell Computational Model Predicts Phenotype from Genotype\nKarr et al. (2012)  "}),a.add({id:18,href:'/SysBioBook/author-info/',title:"About the Author",section:"Home",content:"About the Author #  Karthik Raman is an Associate Professor at the Department of Biotechnology, Bhupat \u0026amp; Jyoti Mehta School of Biosciences, Indian Institute of Technology Madras. He co-founded and co-ordinates the Initiative for Biological Systems Engineering and is a core member of the Robert Bosch Centre for Data Science and Artificial Intelligence (RBCDSAI). He has been a researcher in the area of systems biology for the last 15+ years and has been teaching a course on systems biology for the last eight years, to (mostly) engineers from different backgrounds. His lab works on computational approaches to understand and manipulate biological networks, with applications in metabolic engineering and synthetic biology.\n \n \n \n"}),a.add({id:19,href:'/SysBioBook/errata-feedback/',title:"Errata \u0026 Feedback",section:"Home",content:"Feedback #  Despite my best efforts and those of my colleagues and students who have carefully gone through the book, it is likely that there may be a few typographical/grammatical errors. Do send your feedback via GitHub\u0026mdash;critical feedback will be much appreciated and duly acknowledged!\nErrata #  None so far!\n"}),a.add({id:20,href:'/SysBioBook/faq/',title:"FAQs",section:"Home",content:"Frequently Asked Questions #  What are the pre-requisites for this book?\nThe only pre-requisite for this text is a familiarity with a high-level programming language aside from an intellectual curiosity towards how mathematical and computational approaches aid in understanding biology. An engineering mathematics background will be advantageous but not strictly necessary.  "}),a.add({id:21,href:'/SysBioBook/resources/supplementary-exercises/',title:"Supplementary Exercises",section:"Resources",content:"Supplementary Exercises #  "}),a.add({id:22,href:'/SysBioBook/updates/',title:"Updates",section:"Home",content:"Updates #   2020-10-24: Website is up!  "}),a.add({id:23,href:'/SysBioBook/resources/further-reading/appendix-a/',title:"Appendix A",section:"Further Reading",content:"Chapter A: Introduction to key biological concepts #  "}),a.add({id:24,href:'/SysBioBook/resources/further-reading/appendix-b/',title:"Appendix B",section:"Further Reading",content:"Chapter B: Reconstruction of biological networks #  Reconstruction of biochemical networks in microorganisms\nFeist et al. (2008)  Deciphering Protein-Protein Interactions, Part I (Experimental Techniques)\nShoemaker and Panchenko (2007)  Deciphering Protein-Protein Interactions, Part II (Computational Techniques)\nShoemaker and Panchenko (2007)  Reconstruction of cellular signalling networks and their properties\nPapin et al. (2005)  A survey of metabolic databases emphasizing the MetaCyc family\nKarp and Caspi (2011)  "}),a.add({id:25,href:'/SysBioBook/resources/further-reading/appendix-c/',title:"Appendix C",section:"Further Reading",content:"Chapter C: Databases for systems biology #  "}),a.add({id:26,href:'/SysBioBook/resources/further-reading/appendix-d/',title:"Appendix D",section:"Further Reading",content:"Chapter D: Software tools compendium #  SBGN\n(2009)  "}),a.add({id:27,href:'/SysBioBook/resources/further-reading/appendix-e/',title:"Appendix E",section:"Further Reading",content:"Chapter E: MATLAB for systems biology #  Loren: Art of MATLAB  What Every Computer Scientist Should Know About Floating Point Arithmetic\n()  Is Addition on a Computer Associative?  Wikipedia: Double-precision Floating-point Format  "}),a.add({id:28,href:'/SysBioBook/resources/figure-slides/',title:"Figure Slides",section:"Resources",content:"Figure Slides #  Editable slides in both Powerpoint and PDF (one figure per page) formats are available freely for instructors wishing to use these in their course.\n"})})()