'use strict';(function(){const b={cache:!0};b.doc={id:'id',field:['title','content'],store:['title','href','section']};const a=FlexSearch.create('balance',b);window.bookSearchIndex=a,a.add({id:0,href:'/SysBioBook/resources/appendices/',title:"Appendices",section:"Resources",content:"Appendices #  It is desired to make the book as self-contained as possible; therefore, additional material in the form of five online appendices are also included.\nA. Introduction to Key Biological Concepts #  This appendix provides a quick introduction to biology, focussing on basic biological macromolecules such as nucleic acids, proteins, carbohydrates, lipids, and small molecules. It also provides a brief overview of information flow in biological systems, discussing aspects of DNA replication, mRNA transcription, protein translation, and post-translational modifications.\nB. Reconstruction of Biological Networks #  This appendix provides an overview of reconstruction of various types of biological networks, viz. gene regulatory networks, protein\u0026ndash;protein interaction networks, signalling networks, and metabolic networks.\nC. Databases for Systems Biology #  This appendix presents a brief overview of databases of PPIs and functional associations, metabolic network databases, gene regulatory network databases, model databases, and certain miscellaneous databases.\nD. Software Tools Compendium #  Beginning with a summary of standards and formats, this appendix details various software tools that are useful for all the modelling paradigms studied in the book, such as network biology, dynamic modelling and parameter estimation, Boolean networks, constraint-based modelling, and community modelling.\nE. MATLAB for Systems Biology #  This appendix provides a solid introduction to the basics of using MATLAB, followed by a discussion on how a variety of tools and toolboxes can be used for handling various modelling and simulation tasks in systems biology.\nAppendices are located online-only at https://www.routledge.com/9781138597327.\n"}),a.add({id:1,href:'/SysBioBook/resources/further-reading/chapter-01/',title:"Chapter 1",section:"Further Reading",content:"Chapter 1: Introduction to modelling #  Can a biologist fix a radio?—Or, what I learned while studying apoptosis\nLazebnik (2002) Cancer Cell\nThis insightful paper by Yuri Lazebnik discusses some very interesting points related to engineering vs. biology, and underlines the challenges associated with studying biological complexity.\n From molecular to modular cell biology\nHartwell (1999) Nature\nA nice essay on modularity in biology, and a systems perspective, from Leland Hartwell, who went on to win the 2001 Nobel Prize in Physiology or Medicine.\n Modelling viral and immune system dynamics\nPerelson (2002) Nature Reviews Immunology\nAn authoritative review on modelling the dynamics of viral infections e.g. HIV/HCV.\n Numb3rs 1x03 - Vector (TV episode)\nThe protagonist in this show is a mathematician, who uses SIR modelling to help the FBI track down the spread of a virus in a city.\n Ten simple rules for biologists learning to program\nCarey and Papin (2018)\nA nice guide for folks trying to delve into the world of programming!\n "}),a.add({id:2,href:'/SysBioBook/resources/lecture-videos/chapter-01/',title:"Chapter 1",section:"Lecture Videos",content:"Lecture Videos #  A Massive Open Online Course on Computational Systems Biology, which is in part, a basis for this book, is also available, through the Indian educational platforms, NPTEL (National Programme on Technology Enhanced Learning) and SWAYAM.\nThe lectures (including hands-on lab sessions), totalling over 30 hours of content, are also available on YouTube.\nChapter 1: Introduction to modelling #  Introduction (21:26)\n Introduction What are models? Why model?     Introduction to Modelling (24:03)\n Why model biological systems?     Introduction to Modelling (13:34)\n Examples of Models Challenges in modelling biological systems Discussion: Modelling Spread of Infectious Diseases     Fundamentals of Mathematical Modelling (17:59)\n The Modelling Process Scope / Assumptions Types of Models     Fundamentals of Mathematical Modelling (18:49)\n Model Analysis and Diagnosis Model Applications     Fundamentals of Mathematical Modelling (15:21)\n Example Model: SIR     Some Example Models (5:51)\n   Representation of Biological Networks (8:53)\n How to represent biological networks? SBGN     "}),a.add({id:3,href:'/SysBioBook/resources/supplementary-exercises/chapter-01/',title:"Chapter 1",section:"Supplementary Exercises",content:"Chapter 1: Introduction to modelling #   While studying the metabolic pathways of a cell, the following four reactions were considered to play an important role in cell metabolism,  \rR1: \u0026rightarrow; A\rR2: A \u0026rightarrow; B\rR3: A + B \u0026rightarrow; C\rR4: C \u0026rightarrow; \r But to do computational analyses, these reactions have to be represented mathematically. Can you represent these reactions using a matrix?\nHint: The matrix will have each column representing each reaction and each rows metabolite. If a metabolite m1 is consumed in R1, then the value corresponding to (m1, R1) will be -1. If m2 is produced in R1, then (m2, R1) will be +1. If m3 does not involve in R1, then (m3, R1) will be 0.\na.   \\(\r\\begin{bmatrix}\r1 \u0026amp; -1 \u0026amp; -1 \u0026amp; 0 \\\\\r0 \u0026amp; 1 \u0026amp; -1 \u0026amp; 0 \\\\\r0 \u0026amp; 0 \u0026amp; 1 \u0026amp; -1\r\\end{bmatrix} \\)  b.  \\(\r\\begin{bmatrix}\r1 \u0026amp; -1 \u0026amp; 0 \u0026amp; 0 \\\\\r0 \u0026amp; 1 \u0026amp; 0 \u0026amp; 0 \\\\\r1 \u0026amp; 1 \u0026amp; 1 \u0026amp; -1\r\\end{bmatrix} \\)  c.  \\(\r\\begin{bmatrix}\r-1 \u0026amp; -1 \u0026amp; -1 \u0026amp; 1 \\\\\r0 \u0026amp; 1 \u0026amp; 1 \u0026amp; 0 \\\\\r0 \u0026amp; 0 \u0026amp; -1 \u0026amp; 1\r\\end{bmatrix} \\)  d.  \\(\r\\begin{bmatrix}\r0 \u0026amp; -1 \u0026amp; -1 \u0026amp; 0 \\\\\r1 \u0026amp; -1 \u0026amp; 1 \u0026amp; 0 \\\\\r1 \u0026amp; 0 \u0026amp; 1 \u0026amp; 0\r\\end{bmatrix} \\)  Ans: a\nIn an experiment, the following data was obtained.    Conc. of substrate Conc. of product     1 5   2 10   3 20   4 25   5 25   6 30   7 40   8 60   9 60   10 70       This data was plotted on a graph, and a 6th-degree polynomial equation gave a perfect fit with an  \\( R^2 \\)  value of 0.9933.\nThen they tried to predict the product concentration further by increasing the substrate concentration. The predicted values significantly differed from the experimental values. What might be the reason?\na. This happens for no reason.\nb. The model is underfitting. Hence a better model with a higher  \\(R^2\\)  value must be selected.\nc. The model is overfitting. Therefore, a better model can be chosen even though the  \\(R^2\\)  value is less than that of the current model.\nd. None of the above. Ans: c\n What is the name of the following circuit?\na. Oscillator\nb. Repressilator\nc. Toggle Switch\nd. Amplifier Ans: b\n  The simple biological circuit given below is called a toggle switch.\nA toggle switch consists of two promoters (say P1 and P2) and two repressors (say R1 and R2). R2 represses P1 and R1 represses P2. There are two inducers I1 and I2, inducing P1 and P2, respectively.\nThen based on the above biological circuit, which of the following are true? More than one statement is true.\na. The presence of both IPTG and heat leads to partial GFP production.\nb. The presence of only heat leads to complete GFP production.\nc. The presence of only IPTG leads to complete GFP production.\nd. The presence of only heat does not lead to GFP production.\ne. The presence of only IPTG does not lead to GFP production. Ans: a, c, d\n  The below ODE equations represent a simple SIR model of an X disease.\n \\( \\frac{dS}{dt} = - \\frac{\\beta SI}{N} \\)   \\( \\frac{dI}{dt} = \\frac{\\beta SI}{N} - \\gamma I \\)   \\( \\frac{dR}{dt} = \\gamma I \\)  Where N – total population, S, I, R – people susceptible, infected, recovered, respectively.\nβ – the rate of infection and γ - the rate of recovery.\nWhat are all the assumptions made to generate this model? More than one statement may be correct.\na. The total population varies with time.\nb. The total population is fixed throughout.\nc. There are only three categories of people in this population, susceptible, infected, and recovered.\nd. The infected people would not infect the susceptible.\ne. The recovered people cannot be infected. Ans: b, c, e\n  The rate determining step of Michaelis-Menten kinetics is:\na. The complex dissociation step to produce products\nb. The complex formation step\nc. The product formation step\nd. None of the above Ans: a\n  What is/are the correct interpretations of the image below?\na. The reaction takes place in the cytosol\nb. ATP is required for the reaction to proceed\nc. The reaction is catalysed by pyruvate kinase\nd. All of the above\nAns: a, c\n  "}),a.add({id:4,href:'/SysBioBook/resources/further-reading/chapter-02/',title:"Chapter 2",section:"Further Reading",content:"Chapter 2: Introduction to graph theory #  VAX: Gamifying Epidemic Prevention\nAn engaging game, that uses network theory to model the spread of an epidemic. Players need to stop epidemic spread by vaccinating/quarantining the right \u0026lsquo;nodes\u0026rsquo; (people) from the network.\n Leonhard Euler and the Koenigsberg Bridges\nEuler (1953) Scientific American\nA nice article recapitulating the history of the Koenigsberg bridges.\n The truth about Koenigsberg\nHopkins and Wilson (2018) The College Mathematics Journal\nAnother article on the history of the bridges.\n The SuiteSparse Matrix Collection\nThe SuiteSparse Matrix Collection (formerly the University of Florida Sparse Matrix Collection) is a widely used set of sparse matrix benchmarks collected from a wide range of applications.\n MATLAB Graph Functions Reference\nMATLAB has a suite of functions for studying/analysing networks; these are described here.\n "}),a.add({id:5,href:'/SysBioBook/resources/lecture-videos/chapter-02/',title:"Chapter 2",section:"Lecture Videos",content:"Lecture Videos #  A Massive Open Online Course on Computational Systems Biology, which is in part, a basis for this book, is also available, through the Indian educational platforms, NPTEL (National Programme on Technology Enhanced Learning) and SWAYAM.\nThe lectures (including hands-on lab sessions), totalling over 30 hours of content, are also available on YouTube.\nChapter 2: Introduction to graph theory #  Introduction to Networks (14:31)\n History of Graph Theory Types of Graphs     Introduction to Networks (20:58)\n More Types of Graphs Representing Metabolic Networks Aside: Sparse Matrices     Introduction to Network Biology (25:02)\n Why Graphs? Basic Network Parameters     "}),a.add({id:6,href:'/SysBioBook/resources/supplementary-exercises/chapter-02/',title:"Chapter 2",section:"Supplementary Exercises",content:"Chapter 2: Introduction to graph theory #    A previous class of students had analyzed some data and constructed a small network. All they have left behind for the next batch is an adjacency matrix, given below.\n  \\(\r\\begin{bmatrix}\r0 \u0026amp; 3 \u0026amp; 0 \u0026amp; 0 \u0026amp; 0 \u0026amp; 0 \u0026amp; 0 \\\\\r0 \u0026amp; 0 \u0026amp; 1 \u0026amp; 0 \u0026amp; 0 \u0026amp; 0 \u0026amp; 0 \\\\\r0 \u0026amp; 0 \u0026amp; 0 \u0026amp; 1 \u0026amp; 5 \u0026amp; 0 \u0026amp; 0 \\\\\r0 \u0026amp; 0 \u0026amp; 0 \u0026amp; 0 \u0026amp; 0 \u0026amp; 0 \u0026amp; 10 \\\\\r0 \u0026amp; 0 \u0026amp; 0 \u0026amp; 0 \u0026amp; 0 \u0026amp; 2 \u0026amp; 0 \\\\\r0 \u0026amp; 0 \u0026amp; 0 \u0026amp; 3 \u0026amp; 0 \u0026amp; 0 \u0026amp; 0 \\\\\r0 \u0026amp; 1 \u0026amp; 0 \u0026amp; 0 \u0026amp; 0 \u0026amp; 0 \u0026amp; 0\r\\end{bmatrix} \\)  What do you think the network represents?\na. A weighted, undirected, sparse network.\nb. A weighted, directed, sparse network.\nc. An unweighted, undirected, sparse network.\nd. An unweighted, directed, sparse network.\ne. This information cannot be obtained from the adjacency matrix.\nAns: b\n  Which of the following statements is/are TRUE for undirected graphs?\nA: Number of odd degree vertices is even.\nB: Sum of degrees of all vertices is even.\na. A only\nb. B only\nc. Both A and B\nd. Neither A nor B\nAns: c\n  I was a part of a team whose job was to study a biological network we recently constructed. We felt that properties such as the clustering coefficient, the diameter of the network, the characteristic pathlength all meant something special about the network. Still, we did not know how to verify it. I have the following four options in mind. Help me get a good name amongst my mates by choosing the correct method for me.\na. Build the network again from scratch, and if we get the network with the same properties, they are unique.\nb. Convince teammates; no verification required.\nc. Generate many random networks with the same no. of nodes and edges, check if our network properties are very distinct from 95-100% of random networks generated.\nd. Both a and c.\nAns: c\n  The degree of all nodes would be the same in which of the following networks? More than one statement may be true.\na. Hypergraphs.\nb. Small-world (ring lattice) networks.\nc. Regular ring lattices.\nd. Complete graphs.\ne. An Erdős–Rényi random network with n nodes and the probability of connecting edges as 1.\nAns: a, c, d, e\n  Answer the following questions from the following graph\ni) Degree of node 5:\nAns: 3\nii) Number of connected components:\nAns: 1\niii) Betweenness centrality of node 4:\nAns: 0.45\niv) Average clustering coefficient of the network:\nAns: 0.277\nv) Diameter of a network:\nAns: 3\n  Which of the following is/are NOT a node property? a. Degree\nb. Clustering coefficient\nc. Betweenness centrality\nd. Characteristic path length\nAns: d\n  The local clustering coefficient of a node within a clique is:\na. Infinite\nb. 1\nc. 0\nd. None of the above\nAns: b\n  A graph  \\(G(V, E)\\)  is constructed such that each node is a metabolite and the edges represent reactions by which one metabolite can be converted to another. Every metabolite can be converted into any other metabolite in the graph through a set of reactions. What type of a graph is  \\(G\\)  ?\na. Sparse\nb. Complete\nc. Strongly connected\nAns: c\n  When you delete the node with the highest betweenness centrality, the characteristic path length\na. decreases\nb. increases\nc. does not change\nd. cannot be determined\nAns: b\n  Preferential attachment is a characteristic of\na. Random network\nb. Small world network\nc. Power-law network\nd. All of the above\nAns: c\n  "}),a.add({id:7,href:'/SysBioBook/resources/codes/',title:"Codes",section:"Resources",content:"Codes #  The companion code repository to this book includes some key snippets of code that will enable readers to understand some of the models/methodologies described in this book.\nhttps://github.com/RamanLab/SysBioBook-Codes\nThe idea is to expand this repository over a period of time. If you think there is a piece of code that will help you better understand a concept in the book, please contribute an issue at https://github.com/RamanLab/SysBioBook-Codes/issues and I will be more than happy to include one or more code snippets to address it.\n"}),a.add({id:8,href:'/SysBioBook/resources/further-reading/chapter-03/',title:"Chapter 3",section:"Further Reading",content:"Chapter 3: Structure of networks #  Network Biology\nBarabasi and Oltvai (2004) Nature Reviews Genetics\nA classic review on network biology capturing a very good breadth of topics, ranging from network parameters all the way up to network perturbations.\n How Ideas Spread?\nA very nice article by Duncan Watts, the proponent of the small-world network hypothesis.\n Using graph theory to analyze biological networks\nPavlopoulos et al. (2011) BioData Mining\nA nice review on graph theory applications in network biology.\n Star Wars Social Network! (by Evelina Gabasova)\nA fun exercise, studying the social networks in Star Wars movies!\n Collective dynamics of ‘small-world’ networks\nWatts and Strogatz (1998) Nature\nThe original paper on small world networks.\n Complex networks: Structure and dynamics\nBoccaletti et al. (2006) Physics Reports\nExtremely detailed paper on networks (a mini book/reference!)\n Complex Networks: Small-World, Scale-Free and Beyond\nWang and Chen (2003) IEEE Circuits and Systems Magazine\nA lucid easy-to-read paper discussing network models.\n How Network Theory Is Revealing Previously Unknown Patterns in Sports\nAn interesting read on the use of network science in sports.\n Emergence of Scaling in Random Networks\nBarabasi and Albert (1999) Science\nThe original paper on scale-free networks.\n Structure and Function of Complex Networks\nNewman (2003) SIAM Review\nAnother excellent review/mini-book on networks, covering a wide gamut of topics.\n Network motifs in the transcriptional regulation network of Escherichia coli\nShen-Orr et al. (2002) Nature Genetics\nThe classic paper defining the concept of network motifs.\n Network motifs: Simple building blocks of complex networks\nMilo et al. (2002) Science\nAnother excellent paper building on the concept of network motifs.\n FANMOD: a tool for fast network motif detection\nAn easy-to-use software tool for motif detection.\n Aggregation of topological motifs in the Escherichia coli transcriptional regulatory network\nDobrin et al. (2004) BMC Bioinformatics\nAn insightful analysis of the organisation of E. coli transcriptional regulatory networks: motifs, clusters and superclusters.\n Motifs, themes and thematic maps of an integrated Saccharomyces cerevisiae interaction network\nZhang et al. (2005) Journal of Biology\nDiscusses the concept of ‘network themes’—classes of higher-order recurring interconnection patterns encompassing multiple occurrences of network motifs.\n Error and attack tolerance of complex networks\nBarabasi and Albert (2000) Nature\nThe classic paper illustrating the robustness of scale-free networks to random failures, and susceptibility to targeted attacks.\n Attack vulnerability of complex networks\nHolme et al. (2002) Physical Review E\nAn elaborate analysis of four different attack strategies on a variety of complex networks, based on degree/betweenness centrality.\n "}),a.add({id:9,href:'/SysBioBook/resources/lecture-videos/chapter-03/',title:"Chapter 3",section:"Lecture Videos",content:"Lecture Videos #  A Massive Open Online Course on Computational Systems Biology, which is in part, a basis for this book, is also available, through the Indian educational platforms, NPTEL (National Programme on Technology Enhanced Learning) and SWAYAM.\nThe lectures (including hands-on lab sessions), totalling over 30 hours of content, are also available on YouTube.\nChapter 3: Structure of networks #  Introduction to Network Biology (25:52)\n Centrality Measures Closeness Centrality Betweenness Centrality     Introduction to Network Biology (10:31)\n Connected Components Network Motifs (Quick Intro)     Network Biology (13:11)\n Network Concepts Review     Network Models (8:10)\n Random (\\ER) Networks     Network Models (27:58)\n Small-World (Watts\u0026ndash;Strogatz) Networks Power law Networks     Biological Networks (15:15)\n Centrality\u0026ndash;lethality hypothesis Assortativity     Network Perturbations (14:50)\n Perturbations     Community Detection (21:42)\n Communities in Networks Modularity     Network Motifs (21:53)\n What are Motifs? Randomising a Network Biological significance     Lab: Cytoscape (23:41)\n Cytoscape Introduction STRING Database Loading and Visualising Networks NetworkAnalyzer     Lab: Cytoscape (19:30)\n Simple Network Walkthrough Other Tools     Lab: Network Biology (23:54)\n MATLAB Boost Graph Library (BGL) Regular lattice Rewiring a regular lattice     Network Biology: Recap (37:27)\n Network Parameters Network Models Communities Motifs     Lab: Network Models \u0026amp; Perturbations (7:16)\n Network Models     Lab: Network Models \u0026amp; Perturbations (46:21)\n Network Perturbations Profiling (MATLAB)     "}),a.add({id:10,href:'/SysBioBook/resources/supplementary-exercises/chapter-03/',title:"Chapter 3",section:"Supplementary Exercises",content:"Chapter 3: Structure of networks #    The ______ centrality of a node N1, is the reciprocal of the sum of the shortest distances between N1 and all other nodes.\na. Closeness\nb. Betweenness\nc. Degree\nd. None of the above\nAns: a\n  What is the command for finding the degrees of each node in the network, A is the adjacency matrix?\na. degrees = sum(A)\nb. degrees = nnz(A)\nc. degrees = nonzeros(A)\nd. None.\nAns: a\n  A complex biological graph is typically unaffected by random perturbations because…\na. Random perturbations cancel each other.\nb. Random perturbations always attack peripheral nodes.\nc. Random perturbations rarely affect essential nodes in a scale-free network.\nd. There is no exact reason.\nAns: c\n  Pick the best suitable reason.\nA complete undirected graph has zero network motifs because… (more than one reason might be true)\na. Undirected graphs do not have motifs.\nb. A complete graph is too sparse to have motifs.\nc. Every node is connected to one another, and hence no rewiring is possible.\nd. None of the motifs are statistically significant.\nAns: c, d\n  In a neural network from the brain to ear, all signals from the brain to ear and vice versa should pass through a neuron X. What would this mean?\na. X will have the highest clustering coefficient.\nb. X is the highest degree node in the neural network.\nc. X will have the highest betweenness centrality.\nd. All the above is true.\nAns: c\n  We know that a network model of a biological system helps us to understand the system. But how is rewiring the network helpful? More than one statement might be true.\na. It helps us to distinguish our network from random networks that can be arrived at by chance.\nb. The rewired network should show the same properties as the original network. Only then our network is correct.\nc. Rewiring is done to study the stability of the network.\nd. It helps us to study the statistical significance of our network parameters.\nAns: a, d\n  The information about a network is attached as a text file, which can be uploaded into MATLAB using the command readmatrix(filename, 'FileType', 'text').\nAnswer the following based on this network.\ni) What is the maximum degree of a node in this network?\nAns: 50\nii) How many nodes have a degree of 26?\nAns: 2\niii) How many nodes have all their neighboring nodes interacting with one another?\nAns: 266\niv) What is the characteristic pathlength of this network? Round off to two decimal points.\nAns: 6.50 to 6.51\n  Which of the following gene networks are directed graphs?\na. Gene-regulatory networks\nb. Co-expression networks\nc. Gene interaction networks\nd. Transcriptional regulatory networks\ne. Gene expression network\nAns: a, d\n  Construct a regular lattice in MATLAB with 100 nodes and each node having 10 neighbors. What is the characteristic pathlength of the network? Round off the answer to 2 decimal points.\nAns: 5.4 to 5.5\n  Randomly remove 120 nodes from the regular lattice in Question 9. What will be the mean clustering coefficient of the network? Round off the answer to 2 decimal points.\nAns: 0.3 to 0.44\n  If a network has metabolites and reactions as nodes, with edges between metabolites and reactions, reactions and metabolites it is a _________.\na. Substrate graph\nb. Hypergraph\nc. Bipartite graph\nd. Reaction graph\nAns: c\n  "}),a.add({id:11,href:'/SysBioBook/resources/lecture-videos/',title:"Lecture Videos",section:"Resources",content:"Lecture Videos #  A Massive Open Online Course on Computational Systems Biology, which is in part, a basis for this book, is also available, through the Indian educational platforms, NPTEL (National Programme on Technology Enhanced Learning) and SWAYAM.\nThe lectures (including hands-on lab sessions), totalling over 30 hours of content, are also available on YouTube.\n  Chapter 1   Introduction to modelling   Chapter 2   Introduction to graph theory   Chapter 3   Structure of networks   Chapter 4   Applications of network biology   Chapter 5   Introduction to dynamic modelling   Chapter 6   Parameter estimation   Chapter 7   Discrete dynamic models: Boolean networks   Chapter 8   Introduction to constraint-based modelling   Chapter 9   Extending constraint-based approaches   Chapter 10   Perturbations to metabolic networks   Chapter 11   Modelling cellular interactions   Chapter 12   Designing biological circuits   Chapter 13   Robustness and evolvability of biological systems   Chapter 14   Epilogue: The road ahead   Appendix A   Introduction to key biological concepts   Appendix B   Reconstruction of biological networks   Appendix C   Databases for systems biology   Appendix D   Software tools compendium   Appendix E   MATLAB for systems biology   "}),a.add({id:12,href:'/SysBioBook/resources/further-reading/chapter-04/',title:"Chapter 4",section:"Further Reading",content:"Chapter 4: Applications of network biology #  Lethality and centrality in protein networks\nJeong et al. (2001) Nature\nThe classic paper (with the classic colourful protein interaction network image!) showing how highly connected proteins in a network are much more likely to be essential for (yeast) survival.\n Nine quick tips for analyzing network data\nMiele et al. (2020) PLoS Computational Biology\nA good article emphasising important \u0026ldquo;\u0026ldquo;good practices\u0026rdquo;\u0026rdquo; for analysing network data.\n The human disease network\nGoh et al. (2007) Proceedings of the National Academy of Sciences USA\nAn excellent paper detailing the construction and analysis of bi-partite graphs to study diseases, symptoms, and genes.\n Edgetic perturbation models of human inherited disorders\nZhong et al. (2009) Molecular Systems Biology\nThe classic paper on \u0026ldquo;\u0026ldquo;edgetics\u0026rdquo;\u0026rdquo;.\n Differential network biology\nIdeker and Krogan (2012) Molecular Systems Biology\nThe classic paper describing DiNA, and its application to study some example disease conditions.\n Network-based features enable prediction of essential genes across diverse organisms\nAzhagesan et al. (2018) PLoS ONE\nEmphasises the remarkable utility of network-based features in predicting essential genes in interactomes.\n The organisational structure of protein networks: revisiting the centrality—lethality hypothesis\nRaman et al. (2014) Systems and Synthetic Biology\nRe-examines the centrality—lethality hypothesis from the perspective of essentiality data from DEG, and interactomes from STRING.\n Adapting Community Detection Algorithms for Disease Module Identification in Heterogeneous Biological Networks\nTripathi et al. (2019) Frontiers in Genetics\nDiscusses multiple strategies to identify disease-relevant modules from biological networks.\n NetGenes Database\nPredicts essential genes across 2700+ organisms, capturing local, global and neighbourhood properties of STRING interactomes to extract features.\n Assessment of network module identification across complex diseases\nChoobdar et al. (2019) Nature Methods\nA detailed assessment of numerous algorithms for community detection, as applied to identify disease modules in multiple types of networks.\n Predicting novel metabolic pathways through subgraph mining\nSankar et al. (2017) Bioinformatics\nA method for retrosynthesis that uses only structural information to predict and rank possible biotransformations.\n Protein structure: insights from graph theory\nVishveshwara et al. (2002) Journal of Theoretical and Computational Chemistry\nOne of the classic papers describing the application of graph theory to represent and understand protein structures.\n The Protein Folding Network\nRao and Caflisch (2004) Journal of Molecular Biology\nA detailed study of the network properties of protein folding pathways.\n A General Mechanism for the Propagation of Mutational Effects in Proteins\nRajasekaran et al. (2017) Biochemistry\nA study that integrates data from published NMR experiments, molecular dynamics simulations and network theory to model the allosteric effects of point mutations.\n Toward a quantitative description of microscopic pathway heterogeneity in protein folding\nGopi et al. (2017) Physical Chemistry Chemical Physics\nAn extraordinary paper that applies numerous tools from network theory to shed light on possible microstates in protein folding pathways.\n Missing and spurious interactions and the reconstruction of complex networks\nGuimera and Sales-Pardo (2009) Proceedings of the National Academy of Sciences USA\nA very interesting study on link prediction in networks.\n Network representations and methods for the analysis of chemical and biochemical pathways\nSandefur et al. (2013) Molecular bioSystems\nA nice review on various biological networks and their analyses, including chemical and biochemical reaction networks.\n "}),a.add({id:13,href:'/SysBioBook/resources/lecture-videos/chapter-04/',title:"Chapter 4",section:"Lecture Videos",content:"Lecture Videos #  A Massive Open Online Course on Computational Systems Biology, which is in part, a basis for this book, is also available, through the Indian educational platforms, NPTEL (National Programme on Technology Enhanced Learning) and SWAYAM.\nThe lectures (including hands-on lab sessions), totalling over 30 hours of content, are also available on YouTube.\nChapter 4: Applications of network biology #  Applications of Network Biology (51:46)\n Network Attack Strategies Applications     "}),a.add({id:14,href:'/SysBioBook/resources/supplementary-exercises/chapter-04/',title:"Chapter 4",section:"Supplementary Exercises",content:"Chapter 4: Applications of network biology #  "}),a.add({id:15,href:'/SysBioBook/resources/further-reading/',title:"Further Reading",section:"Resources",content:"Further Reading #  While the book contains over 700 references including special pointers to material for further reading, the following annotated reading list separated chapter-wise will be helpful in digging deeper into the concepts discussed in the text.\n  Chapter 1   Introduction to modelling   Chapter 2   Introduction to graph theory   Chapter 3   Structure of networks   Chapter 4   Applications of network biology   Chapter 5   Introduction to dynamic modelling   Chapter 6   Parameter estimation   Chapter 7   Discrete dynamic models: Boolean networks   Chapter 8   Introduction to constraint-based modelling   Chapter 9   Extending constraint-based approaches   Chapter 10   Perturbations to metabolic networks   Chapter 11   Modelling cellular interactions   Chapter 12   Designing biological circuits   Chapter 13   Robustness and evolvability of biological systems   Chapter 14   Epilogue: The road ahead   Appendix A   Introduction to key biological concepts   Appendix B   Reconstruction of biological networks   Appendix C   Databases for systems biology   Appendix D   Software tools compendium   Appendix E   MATLAB for systems biology   "}),a.add({id:16,href:'/SysBioBook/resources/further-reading/chapter-05/',title:"Chapter 5",section:"Further Reading",content:"Chapter 5: Introduction to dynamic modelling #  An Introduction to Dynamical Systems\nSobie (2011) Science Signaling\nA nice introduction to dynamical systems.\n Quantitative Modeling in Cell Biology: What Is It Good for?\nMogilner et al. (2006) Developmental Cell\nA simple and easy-to-read review, with some lucid examples of dynamic modelling, applied to biology.\n Biochemical Systems Theory: A Review\nVoit (2013) ISRN Biomathematics\nA nice overview of BST (Biochemical Systems Theory).\n "}),a.add({id:17,href:'/SysBioBook/resources/lecture-videos/chapter-05/',title:"Chapter 5",section:"Lecture Videos",content:"Lecture Videos #  A Massive Open Online Course on Computational Systems Biology, which is in part, a basis for this book, is also available, through the Indian educational platforms, NPTEL (National Programme on Technology Enhanced Learning) and SWAYAM.\nThe lectures (including hands-on lab sessions), totalling over 30 hours of content, are also available on YouTube.\nChapter 5: Introduction to dynamic modelling #  Introduction to Dynamic Modelling (10:54)\n Dynamic Modelling     Introduction to Dynamic Modelling (22:54)\n Michaelis\u0026ndash;Menten Kinetics An Example System     Introduction to Dynamic Modelling (9:11)\n Solving ODEs     Lab: Solving ODEs in MATLAB (19:50)\n Simple iterative code ODE solvers in MATLAB (e.g. ode15s)     Lab: Example Biological Model (17:42)\n Sustained Oscillations in Glycolysis     "}),a.add({id:18,href:'/SysBioBook/resources/supplementary-exercises/chapter-05/',title:"Chapter 5",section:"Supplementary Exercises",content:"Chapter 5: Introduction to dynamic modelling #    For the case where continuum of molecules cannot be approximated, rate of reaction is proportional to:\na. reactant concentrations\nb. product concentrations\nc. reactant partial pressures\nd. probability of collision of reactants\nAns: d\n  In Hill’s equation, which is given as:   \\(v =\\frac{v_{max}S^n}{k\u0026#39;_m\u0026#43;S^n} \\)  , what does  \\(n \u0026gt;1 \\)  indicate?\na. Competition\nb. Cooperation\nc. Deactivation\nd. None of these\nAns: b\n  A and B are two equations defined as follows, based on the two Euler methods:\n(A)  \\(x_2 = x_1 \u0026#43; f(x_1,t_1 )h\\)  (B)  \\(x_2 = x_1 \u0026#43; f(x_2,t_2 )h\\)  If I say that the equation (B) is (i) implicit and can be solved using (ii) Newton-Raphson method, choose the option to which you agree.\na. (i) is true and (ii) is true\nb. (i) is true and (ii) is false\nc. (i) is false and (ii) is true\nd. (i) is false and (ii) is false\nAns: a\n  Solve the following ODEs using ode15s in MATLAB\n \\(\\frac{dy_1}{dt}= y_2\\)   \\(\\frac{dy_2}{dt}= μ(1-y_1^2 ) y_2- y_1\\)   \\(μ = 1000, y(t=0) = [2;0]\\)  Report the value of  \\(y_1 ( t = 3000)\\)  .\nAns: -1.502\n  Can you solve the above ODE system (of Q.4) with ode45? Can you solve it when the parameter μ = 1? What do you think about the stiffness of the ODEs?\na. The ODEs are stiff when μ = 1000\nb. The ODEs are non-stiff when μ = 1000\nAns: a\n  "}),a.add({id:19,href:'/SysBioBook/resources/further-reading/chapter-06/',title:"Chapter 6",section:"Further Reading",content:"Chapter 6: Parameter estimation #  Linking data to models: data regression\nJaqaman and Danuser (2006) Nature Reviews Molecular Cell Biology\nAn excellent reference on parameter estimation / regression that makes many critical points.\n Good practices for building dynamical models in systems biology\nAzeloglu and Iyengar (2015) Science Signaling\nA critical analysis of possible pitfalls in dynamical modelling, with a detailed flowchart for rigorous model development.\n Reverse engineering and identification in systems biology: strategies, perspectives and challenges\nVillaverde and Banga (2014) Journal of the Royal Society Interface\nA nice survey of some important topics related to reverse engineering and system identification in systems biology.\n Recent developments in parameter estimation and structure identification of biochemical and genomic systems\nChun-Chou and Voit (2009) Mathematical Biosciences\nA useful overview of parameter estimation in biological systems.\n Parameter Estimation in Biochemical Pathways: A Comparison of Global Optimization Methods\nMoles et al. (2003) Genome Research\nA critical comparison of methods for parameter estimation.\n Programming biological models in Python using PySB\nLopez et al. (2013) Molecular Systems Biology\nDescribes PySB, a new approach to create models as programs.\n Dynamical modeling and multi-experiment fitting with PottersWheel\nMaiwald and Timmer (2008) Bioinformatics\nPottersWheel is a MATLAB toolbox for numerically fast mechanistic mathematical modeling, and can be applied to any ODE system.\n Evolutionary Computation\nFoster (2001) Nature Reviews Genetics\nAn excellent overview of evolutionary computation.\n Sensitivity analysis approaches applied to systems biology models\nZi (2011) IET Systems Biology\nA very nice overview of sensitivity analysis for systems biology.\n An Introduction to Genetic Algorithms, by Melanie Mitchell (1995)\nA lucid introduction to genetic algorithms.\n GAs for Travelling Salesperson Problem\nDiscusses GAs to solve TSP, including a tutorial with interactive Java Applets.\n "}),a.add({id:20,href:'/SysBioBook/resources/lecture-videos/chapter-06/',title:"Chapter 6",section:"Lecture Videos",content:"Lecture Videos #  A Massive Open Online Course on Computational Systems Biology, which is in part, a basis for this book, is also available, through the Indian educational platforms, NPTEL (National Programme on Technology Enhanced Learning) and SWAYAM.\nThe lectures (including hands-on lab sessions), totalling over 30 hours of content, are also available on YouTube.\nChapter 6: Parameter estimation #  Parameter Estimation (12:45)\n Biochemical Systems Theory Optimisation: What is a good fit?     Parameter Estimation (16:59)\n Setting up the Optimisation Problem     Parameter Estimation (20:06)\n Data-driven Mechanistic Modelling: Flowchart     Methods for Parameter Estimation (23:57)\n Introduction Generic Recipe for Direct Search Algorithms     Direct Search Methods (23:14)\n Stochastic Search Algorithms Simulated Annealing Genetic Algorithms     Genetic Algorithms (12:14)\n Other Applications for GAs     Genetic Algorithms (21:02)\n Representation Challenges \u0026amp; Examples Some Illustrations     Other Evolutionary Algorithms (21:24)\n Differential Evolution Implementing EAs When to use EAs? Curse of Dimensionality     PyGMO (16:00)\n Introduction to PyGMO PyGMO Algorithms and Examples Multi-objective Optimisation     Dynamic Modelling Recap (11:58)\n   Lab: Parameter Estimation (30:36)\n Computing the Objective Function \\texttt{fminsearch}     "}),a.add({id:21,href:'/SysBioBook/resources/supplementary-exercises/chapter-06/',title:"Chapter 6",section:"Supplementary Exercises",content:"Chapter 6: Parameter estimation #    Solve the following ODE in MATLAB using ode45 and report the value of   \\(y(t=10)\\)  . Report 4 significant digits of your answer.\n \\[\r\\frac{dy}{dt} = \\frac{y\u0026#43;t^2-2}{t\u0026#43;1}; y(0) = 2\r\\]  Ans: 69. 25\n  Using fminsearch in MATLAB, find the minimum of the following function.  \\[ f(x,y) = x^2y - 2xy^2 \u0026#43;3xy \u0026#43;4\r\\]  Use  \\([-1.2, 1]\\)  as the initial guess. Report 2 significant digits of all the values.\n Report the minimum value.\nAns: 3.5 Report the value of xmin.\nAns: -1.0 Report the value of ymin.\nAns: 0.5    I have a measured set of data points as given in the following table.    x y     500 0.05   550 0.1   600 0.2   650 0.4   700 0.8    It is known that y and x are related according to the equation:  \\(y=A \\exp(\\frac{−B}{x})\\)  . Which of the following values of the parameters fit the measured data points well?\na. A = 1000, B = 5000\nb. A = 900, B = 4500\nc. A = 5000, B = 5500\nAns: a\n  "}),a.add({id:22,href:'/SysBioBook/resources/further-reading/chapter-07/',title:"Chapter 7",section:"Further Reading",content:"Chapter 7: Discrete dynamic models: Boolean networks #  Modelling and analysis of gene regulatory networks\nKarlebach and Shamir (2008) Nature Reviews Molecular Cell Biology\nAn excellent review detailing multiple methods for modelling gene regulatory networks.\n Boolean network simulations for life scientists\nAlbert et al. (2008) Source Code for Biology and Medicine\nDescribes the BooleanNet tool.\n Boolean Modeling of Genetic Regulatory Networks\nAlbert et al. (2004) Lecture Notes in Physics\nA classic paper on modelling genetic regulatory networks, with application to Drosophila melanogaster segment polarity genes.\n Robustness and fragility of Boolean models for genetic regulatory networks\nChaves et al. (2005) Journal of Theoretical Biology\nIntegrates stochasticity nicely, into Boolean modelling.\n Algebraic Models of Biochemical Networks\nLaubenbacher and Jarrah (2009) Methods in Enzymology\nDiscusses generalisations of Boolean networks (algebraic models).\n Dynamical analysis of a generic Boolean model for the control of the mammalian cell cycle\nFaure et al. (2006) Bioinformatics\nA lucid paper, explaining how the cell cycle can be well-modelled using Boolean networks.\n BooleSim\nBooleSim tool, which can be tinkered around with just a web browser.\n BooleanNet\nBooleanNet, a Python-based tool for Boolean Network analysis.\n Predicting Essential Components of Signal Transduction Networks: A Dynamic Model of Guard Cell Abscisic Acid Signaling\nLi et al. (2006) PLoS Biology\nAnother excellent illustration of using Boolean networks to model cellular signalling/regulation.\n "}),a.add({id:23,href:'/SysBioBook/resources/lecture-videos/chapter-07/',title:"Chapter 7",section:"Lecture Videos",content:"Lecture Videos #  A Massive Open Online Course on Computational Systems Biology, which is in part, a basis for this book, is also available, through the Indian educational platforms, NPTEL (National Programme on Technology Enhanced Learning) and SWAYAM.\nThe lectures (including hands-on lab sessions), totalling over 30 hours of content, are also available on YouTube.\nChapter 7: Discrete dynamic models: Boolean networks #  Modelling Gene Regulatory Networks (13:47)\n Logical Models: Boolean Networks     Modelling Gene Regulatory Networks (15:59)\n Examples     Modelling Gene Regulatory Networks (14:23)\n Continuous Models     Lab: Modelling Gene Regulatory Networks (21:59)\n BooleSim (JavaScript)     Lab: Modelling Gene Regulatory Networks (16:47)\n BooleanNet (Python)     "}),a.add({id:24,href:'/SysBioBook/resources/supplementary-exercises/chapter-07/',title:"Chapter 7",section:"Supplementary Exercises",content:"Chapter 7: Discrete dynamic models: Boolean networks #   Consider the following reaction sequence and choose the correct statement(s) about the essentiality of the genes, g1, g2 and g3.   R1: A + B \u0026rightarrow; C g1 OR g2 R2: C + A \u0026rightarrow; D g2 OR g3 R3: B + D \u0026rightarrow; E + F g1 AND g2 R4: A + D \u0026rightarrow; E + F g1 AND g2 R5: E + F \u0026rightarrow; Biomass   a. g1 is non-essential\nb. g2 is non-essential\nc. g3 is non-essential\nd. g1, g2 and g3 are all essentials\nAns: c\nThe following figure shows a simple gene regulatory network. Normal arrowheads on the edges indicate activation and circle-headed blue arrows indicate inhibition/repression. Check the correct Boolean transfer function(s).    a. g1* = g4 AND (NOT g2)\nb. g2* = g1 AND (NOT g3)\nc. g3* = NOT g4\nd. g2* = g3 AND (NOT g1)\n Ans: a, b\n"}),a.add({id:25,href:'/SysBioBook/resources/further-reading/chapter-08/',title:"Chapter 8",section:"Further Reading",content:"Chapter 8: Introduction to constraint-based modelling #  A protocol for generating a high-quality genome-scale metabolic reconstruction\nThiele and Palsson (2010) Nature Protocols\nThe classic paper discussing a detailed 96-step protocol for careful manual curation and reconstruction of a constraint-based model.\n What is flux balance analysis?\nOrth et al. (2010) Nature Biotechnology\nA classic pedagogical piece introducing flux balance analysis.\n Advances in flux balance analysis\nKauffman et al. (2003) Current Opinion in Biotechnology\nThe classic review on FBA.\n Flux balance analysis of biological systems: applications and challenges\nRaman and Chandra (2009) Briefings in Bioinformatics\nA simple review on constraint-based modelling approaches, with an illustration of a simple FBA.\n "}),a.add({id:26,href:'/SysBioBook/resources/lecture-videos/chapter-08/',title:"Chapter 8",section:"Lecture Videos",content:"Lecture Videos #  A Massive Open Online Course on Computational Systems Biology, which is in part, a basis for this book, is also available, through the Indian educational platforms, NPTEL (National Programme on Technology Enhanced Learning) and SWAYAM.\nThe lectures (including hands-on lab sessions), totalling over 30 hours of content, are also available on YouTube.\nChapter 8: Introduction to constraint-based modelling #  Constraint-based Modelling of Metabolic Networks (22:53)\n Introduction Constraints on Biological Systems Stoichiometric Matrix     Flux Balance Analysis (21:21)\n Steady-state Mass Balance Constraints Optimisation (LP)     Flux Balance Analysis (21:39)\n The Objective Function Illustration of FBA     Flux Balance Analysis (17:58)\n Choice of Objective Function Alternate Optima in FBA     Lab: FBA using MATLAB (24:19)\n Understanding and Solving FBA using \\texttt{linprog} COBRA Model Structure FBA using COBRA Toolbox     Lab: COBRA Toolbox (46:28)\n \\texttt{optimizeCbModel} Exchange Reactions Varying Glucose Uptake Rates     Understanding FBA (15:08)\n Gaps Dead-end Metabolites Blocked Reactions     Understanding FBA (16:34)\n Troubleshooting FBA ATP Maintenance Flux Flux Variability Analysis     "}),a.add({id:27,href:'/SysBioBook/resources/supplementary-exercises/chapter-08/',title:"Chapter 8",section:"Supplementary Exercises",content:"Chapter 8: Introduction to constraint-based modelling #   Match the constraints given in the table below with their physical significance.\na. A – ii, B – i, C – iii\nb. A – iii, B – i, C – ii\nc. A – iii, B – ii, C – i\nd. A – i, B – ii, C – i     I II     (A) Stoichiometry (i) Thermodynamic feasibility and equilibrium   (B) Reversibility (ii) Turnover number of involved enzyme   (C) Capacity (iii) Conservation of mass     Ans: b\n Which of the following statements is/are correct?\na. One flux balance equation is written for each metabolite present in the system.\nb. One flux balance equation is written for each reaction occurring in the system.\nc. Metabolites constitute the rows and reactions constitute the columns of the stoichiometric matrix.\nd. Generally in a biochemical system, we can find unique flux distribution for metabolic reactions.\nAns: a, c\n  Download the iAF1260 model from BiGG database and load it in MATLAB. A structure named ‘iAF1260’ will form in the workspace. Use linprog to find the maximum biomass value for this model.\nAns: 0.7367\n  The function linprog in MATLAB does minimization of the objective function by default. Since you want to use it for maximizing the growth rate, you would pass   \\(-c^{T}v\\)  as the objective function. So, the optimum value returned by linprog is actually the minimum of  \\(-c^{T}v\\)  . To obtain the required values of maximum biomass growth rate and the flux distribution, I say you should (i) multiply -1 to the optimum value and (ii) multiply -1 to all the flux values.\nWhat do you think about my statements?\na. Both (i) and (ii) are true\nb. Both (i) and (ii) are false\nc. (i) is false, (ii) is true\nd. (i) is true, (ii) is false\nAns: d\n  Match the following in the context of constraint based metabolic modelling :-\nr: number of reactions, m: number of metabolites\n     I II III     (A.i) m\u0026gt;r (B.i) Unique solution (C.i) Least squares   (A.ii) r\u0026gt;m (B.ii) Under-determined (C.ii) Many possible flux distributions   (A.iii) r=m (B.iii) Over-determined (C.iii) Trivial solution     a. A.i – B.i – C.i; A.ii – B.ii – C.ii; A.iii – B.iii – C.iii\nb. A.i – B.iii – C.i; A.ii - B.ii – C.ii; A.iii – B.i – C.iii\nc. A.i – B.ii – C.i; A.ii – B.i – C.iii; A.iii – B.iii – C.ii\nd. A.i – B.i – C.ii; A.ii – B.iii – C.i; A.iii – B.ii – C.iii\nAns: b\n The figure below depicts the metabolic network of a fictitious organism. It can grow on two substrates (A \u0026amp; C). The pathways utilizing the two substrates are as shown in the figure and G is an indication of the biomass. A and C are only uptaken and not secreted, i.e., the exchange fluxes are only in the direction indicated in the figure. The cell tries to maximize the flux of G. Which of the following statements is/are true?\na. This network is blocked when only A is uptaken. However, it can grow when C is provided and hence this network is “conditionally” blocked.\nb. This network is unconditionally blocked and will not show growth in the presence of any substrate.\nc. When FBA is done on this network, except the biomass flux, every other flux will have a non-zero value.\nd. When FBA is done on this network, all the fluxes will be 0.\nAns: b, d\n  If you conduct FVA for the case considered in question 6, what do you expect?\na. vmin = vmax = 0 for all fluxes\nb. vmin = 0 and vmax ≠ 0 for all fluxes\nc. vmin ≠ 0 and vmax = 0 for all fluxes\nd. vmin ≠ 0 and vmax ≠ 0 for all fluxes\nAns: a\n  "}),a.add({id:28,href:'/SysBioBook/resources/further-reading/chapter-09/',title:"Chapter 9",section:"Further Reading",content:"Chapter 9: Extending constraint-based approaches #  Analysis of optimality in natural and perturbed metabolic networks\nSegre et al. (2002) Proceedings of the National Academy of Sciences USA\nThe classic paper on quadratic programming applied to COBRA: Minimisation of Metabolic Adjustment (MoMA).\n Using Genome-scale Models to Predict Biological Capabilities\nO' Brien et al. (2015) Cell\nAn excellent review covering various applications of metabolic modelling.\n Regulatory on-off minimization of metabolic flux changes after genetic perturbations\nShlomi et al. (2005) Proceedings of the National Academy of Sciences USA\nThis paper describes ROOM, an MILP-based method for constraint-based analysis of metabolic networks.\n In Silico Identification of Gene Amplification Targets for Improvement of Lycopene Production\nChoi et al. (2010) Applied and Environmental Microbiology\nThis paper describes FSEOF, for identifying over-expression targets in metabolic networks, for metabolic engineering.\n Computational identification of gene over-expression targets for metabolic engineering of taxadiene production\nBoghigian et al. (2012) Applied Microbiology and Biotechnology\nA nice illustration of computational approaches to identify over-expression targets for metabolic engineering.\n Predicting Gene Essentiality Using Genome-Scale in silico Models\nJoyce and Palsson (2008) Methods in Molecular Biology\nA nice review, detailing the strengths and limitations of constraint-based modelling, especially for predicting gene deletions/phenotypes.\n "}),a.add({id:29,href:'/SysBioBook/resources/lecture-videos/chapter-09/',title:"Chapter 9",section:"Lecture Videos",content:"Lecture Videos #  A Massive Open Online Course on Computational Systems Biology, which is in part, a basis for this book, is also available, through the Indian educational platforms, NPTEL (National Programme on Technology Enhanced Learning) and SWAYAM.\nThe lectures (including hands-on lab sessions), totalling over 30 hours of content, are also available on YouTube.\nChapter 9: Extending constraint-based approaches #  Other Constraint-Based Approaches (17:41)\n Minimisation of Metabolic Adjustment     Other Constraint-Based Approaches (12:02)\n Regulatory On-Off Minimisation     Elementary Modes (16:19)\n Definitions Simple Examples     Elementary Modes (13:09)\n Another Example     $^{13}$C-Metabolic Flux Analysis using Mass Spectrometry (11:27)\n Introduction to $^{13}C$-MFA     $^{13}$C-Metabolic Flux Analysis using Mass Spectrometry (11:55)\n Methodology GC-MS Analysis     $^{13}$C-Metabolic Flux Analysis using Mass Spectrometry (12:31)\n Data Analysis An Example     Lab: $^{13}$C-Metabolic Flux Analysis using Mass Spectrometry (32:27)\n Recap Simple Exercises     "}),a.add({id:30,href:'/SysBioBook/resources/supplementary-exercises/chapter-09/',title:"Chapter 9",section:"Supplementary Exercises",content:"Chapter 9: Extending constraint-based approaches #    After deleting a gene from a metabolic model, you decide to use MoMA to find the new flux distribution using the function quadprog in MATLAB. The command you have to enter is :-   \\[ x = quadprog(H, f, A, b, A_{eq} , b_{eq}) \\]  Similar to the case in linprog, A and b are null vectors and  \\(A_{eq} = S\\)  and  \\(b_{eq} = 0\\)  . What are the values of H and f that you would pass?\na.  \\(H = -c; f = []\\)  b.  \\(H = 0.5* v^{T} I v; f = -v_w^{T} v \\)  c.  \\(H = I ; f = -v_w \\)  d. None of these\nAns: c\n  Which of the following statements are true about ROOM?\na. ROOM minimizes the number of significant flux changes from the wild type\nb. ROOM minimizes the  \\(L_{2}\\)  norm of the flux difference from the wild type\nc. ROOM provides accurate flux predictions of the metabolic steady state after adaptation\nd. ROOM provides more accurate transient metabolic states following perturbation\nAns: a, c\n  Which of the following statements are correct?\na. Extreme pathways (EP) are an independent subset of Elementary Flux Modes (EFM)\nb. EP can be decomposed into the linear combinations of EFM\nc. EFM can be decomposed into the linear combinations of EP\nAns: a, c\n  Pick the correct statement which explains the inherent assumption of the standard steady-state 13C-MFA\na. The cells are assumed to be under metabolic steady-state, and sampling is done during the isotopic non-steady state.\nb. The cells are assumed to be in a dynamic metabolic state, and sampling is done during the isotopic non-steady state.\nc. The cells are assumed to be in a dynamic metabolic state, and sampling is done once the isotopic steady-state is achieved.\nd. The cells are assumed to be under metabolic steady-state, and sampling is done once the isotopic steady-state is achieved.\nAns: d\n  "}),a.add({id:31,href:'/SysBioBook/resources/further-reading/chapter-10/',title:"Chapter 10",section:"Further Reading",content:"Chapter 10: Perturbations to metabolic networks #  Fast-SL: an efficient algorithm to identify synthetic lethal sets in metabolic networks\nPratapa et al. (2015) Bioinformatics\nDescribes a novel and rapid method for identifying synthetic lethals in metabolic networks, by cutting through the search space.\n Genome-scale gene/reaction essentiality and synthetic lethality analysis\nSuthers et al. (2009) Molecular Systems Biology\nDescribes SL-Finder, an MILP-based method to identify synthetic lethals.\n Enumeration of Smallest Intervention Strategies in Genome-Scale Metabolic Networks\nvon Kamp and Klamt (2014) PLoS Computational Biology\nDescribes MCSEnumerator, an efficient MILP-based method to identify synthetic lethals.\n Global reconstruction of the human metabolic network based on genomic and bibliomic data\nDuarte et al. (2007) Proceedings of the National Academy of Sciences USA\nThe classic reconstruction of human metabolic network, \u0026ldquo;\u0026ldquo;Recon 1\n A community-driven global reconstruction of human metabolism\nThiele et al. (2013) Nature biotechnology\nThe Recon 2 human metabolic model.\n Computational reconstruction of tissue-specific metabolic models: application to human liver metabolism\nJerby et al. (2010) Molecular Systems Biology\nTissue-specific metabolic reconstruction.\n Insight into human alveolar macrophage and M. tuberculosis interactions via metabolic reconstructions\nBordbar et al. (2010) Molecular Systems Biology\nModelling host–pathogen interactions using metabolic modelling.\n A multi-tissue type genome-scale metabolic network for analysis of whole-body systems physiology\nBordbar et al. (2011) BMC Systems Biology\nTissue-specific metabolic reconstruction, for Recon 1, modelling adipocytes, hepatocytes, and myocytes.\n Reconstruction of Genome-Scale Active Metabolic Networks for 69 Human Cell Types and 16 Cancer Types Using INIT\nAgren et al. (2012) PLoS Computational Biology\nExcellent example of context-specific metabolic reconstructions.\n Predicting selective drug targets in cancer through metabolic networks\nFolger et al. (2011) Molecular Systems Biology\nThe first genome-scale metabolic model of cancer metabolism.\n Large-scale in silico modeling of metabolic interactions between cell types in the human brain\nLewis et al. (2010) Nature Biotechnology\nA workflow that integrates gene expression data, proteomics data and literature-based manual curation to model human metabolism within and between different types of cells.\n A compendium of inborn errors of metabolism mapped onto the human metabolic network\nSahoo et al. (2012) Molecular bioSystems\nA comprehensive knowledgebase of inborn errors of metabolism.\n Predicting the impact of diet and enzymopathies on human small intestinal epithelial cells\nSahoo and Thiele (2013) Human Molecular Genetics\nThis paper defined fifty metabolic tasks to evaluate model functionalities.\n How Will Bioinformatics Influence Metabolic Engineering?\nEdwards and Palsson (1997) Biotechnology and Bioengineering\nA classic review, discussing computational approaches for metabolic engineering.\n Basic concepts and principles of stoichiometric modeling of metabolic networks\nMaarleveld et al. (2013) Biotechnology Journal\nAn excellent review covering the breadth of metabolic modelling.\n The Convex Basis of the Left Null Space of the Stoichiometric Matrix Leads to the Definition of Metabolically Meaningful Pools\nFamili and Palsson (2003) Biophysical Journal\nA nice paper providing detailed insights into stoichiometric matrix and the convex basis of the metabolic solution spaces.\n Systemic metabolic reactions are obtained by singular value decomposition of genome-scale stoichiometric matrices\nFamili and Palsson (2003) Journal of Theoretical Biology\nSVD analysis of the stoichiometric matrix.\n Constraining the metabolic genotype–phenotype relationship using a phylogeny of in silico methods\nLewis et al. (2012) Nature Reviews Microbiology\nA detailed review of the entire family of COBRA methods (with a beautiful method phylogeny picture).\n Regulation of Gene Expression in Flux Balance Models of Metabolism\nCovert et al. (2001) Journal of Theoretical Biology\nThe classic paper on rFBA.\n Interpreting Expression Data with Metabolic Flux Models: Predicting Mycobacterium tuberculosis Mycolic Acid Production\nColijn et al. (2009) PLoS Computational Biology\nDescribes the E-Flux method for integration of regulatory information.\n Probabilistic integrative modeling of genome-scale metabolic and regulatory networks in Escherichia coli and Mycobacterium tuberculosis\nChandrasekharan and Price (2010) Proceedings of the National Academy of Sciences USA\nDescribes the PROM approach for integration of regulatory information from gene expression datasets.\n Constraint-based models predict metabolic and associated cellular functions\nBordbar et al. (2014) Nature Reviews Genetics\nAn excellent review on constraint-based models and their applications.\n Identifying gene targets for the metabolic engineering of lycopene biosynthesis in Escherichia coli\nAlper et al. (2005) Metabolic Engineering\nA classic application of FBA for metabolic engineering, including identification of a triple knock-out for improving lycopene production.\n Flux Balance Analysis of Mycolic Acid Pathway: Targets for Anti-Tubercular Drugs\nRaman et al. (2005) PLoS Computational Biology\nA classic application of FBA for drug target identification by modelling the mycolic acid pathway in Mycobacterium tuberculosis.\n targetTB: A target identification pipeline for Mycobacterium tuberculosis through an interactome, reactome and genome-scale structural analysis\nRaman et al. (2008) BMC Systems Biology\nA rigorous pipeline for drug target identification in Mycobacterium tuberculosis.\n Applications of genome-scale metabolic reconstructions\nOberhardt et al. (2009) Molecular Systems Biology\nA detailed review, discussing various classes of applications of constraint-based modelling.\n The application of flux balance analysis in systems biology\nGianchandani et al. (2010) Wiley Interdisciplinary Reviews: Systems Biology and Medicine\nA nice and simple overview of FBA and related methods, with applications.\n Genome-scale metabolic networks\nTerzer et al. (2009) Wiley Interdisciplinary Reviews: Systems Biology and Medicine\nA very nice review on various aspects of metabolic network reconstruction and simulation.\n Elementary flux modes in a nutshell: Properties, calculation and applications\nZanghellini et al. (2013) Biotechnology Journal\nA neat introduction to EFM analysis, using a \u0026ldquo;\u0026ldquo;metro map\u0026rdquo;\u0026rdquo; analogy.\n Detection of elementary flux modes in biochemical networks: a promising tool for pathway analysis and metabolic engineering\nSchuster et al. (1999) Trends in Biotechnology\nAn excellent paper on EFMs.\n Elementary mode analysis: a useful metabolic pathway analysis tool for characterizing cellular metabolism\nTrinh et al. (2009) Applied microbiology and biotechnology\nA nice review detailing EFMs, especially their applications for metabolic engineering.\n Comparison of network-based pathway analysis methods\nPapin et al. (2004) Trends in Biotechnology\nA neat illustration of EFM vs. EPA analyses.\n On Elementary Flux Modes in Biochemical Reaction Systems at Steady State\nSchuster and Hilgetag (1994) Journal of Biological Systems\nThe classic paper on EFMs.\n Extreme Pathway Analysis of Human Red Blood Cell Metabolism\nWiback and Palsson (2002) Biophysical journal\nDiscusses EPA, for studying human RBC metabolism.\n High-throughput metabolic flux analysis based on gas chromatography�mass spectrometry derived 13C constraints\nFischer et al. (2004) Analytical Biochemistry\nA classic paper giving a clear rundown of 13C-MFA.\n 13C Metabolic Flux Analysis\nWiechert (2001) Metabolic Engineering\nA classic review on 13C-MFA.\n 13C-based metabolic flux analysis\nZamboni et al. (2009) Nature Protocols\nA detailed protocol paper elucidating the 13C-MFA approach.\n How to measure metabolic fluxes: a taxonomic guide for 13C fluxomics\nNiedenfuehr et al. (2015) Current Opinion in Biotechnology\nAnother excellent review on 13C-MFA.\n 13C Metabolic Flux Analysis for Systematic Metabolic Engineering of S. cerevisiae for Overproduction of Fatty Acids\nGhosh et al. (2016) Frontiers in Bioengineering and Biotechnology\nA nice illustration of flux analysis for improving production of fatty acids in yeast.\n "}),a.add({id:32,href:'/SysBioBook/resources/lecture-videos/chapter-10/',title:"Chapter 10",section:"Lecture Videos",content:"Lecture Videos #  A Massive Open Online Course on Computational Systems Biology, which is in part, a basis for this book, is also available, through the Indian educational platforms, NPTEL (National Programme on Technology Enhanced Learning) and SWAYAM.\nThe lectures (including hands-on lab sessions), totalling over 30 hours of content, are also available on YouTube.\nChapter 10: Perturbations to metabolic networks #  Perturbations to Metabolic Networks: Deletions (12:50)\n Gene Deletions Reaction Deletions Gene-Protein-Reaction Associations     Perturbations to Metabolic Networks: Over-expression (15:36)\n FSEOF     Perturbations to Metabolic Networks: Synthetic Lethals (13:07)\n Synthetic Lethals     Perturbations to Metabolic Networks: Synthetic Lethals (16:16)\n Fast-SL Algorithm     Constraint-based Modelling of Metabolic Networks (10:54)\n Limitations of Constraint-Based Approaches     Lab: Gene Deletions (30:39)\n Gene Deletions using COBRA Toolbox Escher Single and Double Deletions     Integrating Regulatory Information into Constraint-Based Models (17:53)\n rFBA E-Flux     Constraint-based Modelling of Metabolic Networks: Applications (15:04)\n Introduction Drug Target Identification: Mycolic Acid Pathway in TB     Constraint-based Modelling of Metabolic Networks: Applications (8:30)\n Metabolic Engineering: Lycopene Biosynthesis     Constraint-based Modelling of Metabolic Networks: Applications (17:12)\n targetTB     Lab: Gene Deletions (25:25)\n Single Reaction Deletion Minimal Reactome     Constraint-based Modelling of Metabolic Networks: Recap (16:03)\n Stoichiometric Matrix Constraints Choice of Objective Function     Constraint-based Modelling of Metabolic Networks: Recap (17:57)\n Perturbations     Constraint-based Modelling of Metabolic Networks: Recap (15:34)\n FSEOF Synthetic Lethals     "}),a.add({id:33,href:'/SysBioBook/resources/supplementary-exercises/chapter-10/',title:"Chapter 10",section:"Supplementary Exercises",content:"Chapter 10: Perturbations to metabolic networks #    Consider the following reaction sequence and choose the correct statement(s) about synthetic and single lethals.   \\[ R1: A \u0026#43; B \\rightarrow C \\]   \\[ R2: C \u0026#43;A \\rightarrow D \\]   \\[ R3: B \u0026#43; D \\rightarrow E \u0026#43; F \\]   \\[ R4: A \u0026#43; D \\rightarrow E \u0026#43; F \\]   \\[ R5: E \u0026#43; F \\rightarrow Biomass \\]  a. R3 and R4 is a synthetic lethal pair\nb. R2 and R4 is a synthetic lethal pair\nc. R3 is a single lethal\nd. R4 is a single lethal\nAns: a, b\nDownload the Escherichia coli str. K-12 substr. MG1655 model ‘iAF1260.mat’ from http://bigg.ucsd.edu/models/iAF1260. Load and analyze the model in MATLAB.\n  What is the number of essential reactions (growth rate upon deletion results in \u0026lt; 5% of the wildtype growth) observed in the anaerobic iAF1260 model?\nAns: 292\n  Perform a knockout of the Pyruvate Dehydrogenase (PDH) reaction in the anaerobic model of iAF1260. What is the maximum acetate production flux of the knockout strain (The biomass flux should at least 90% of the wildtype under anaerobic condition)?\nAns: 8.26 mmol/gDW/h\n  Which of the following can help to identify a potential drug target for a bacterial disease affecting humans?\na. A gene that does not change the biomass flux of the bacterium when knocked out.\nb. A gene that does that significantly reduces the biomass flux of the bacterium when knocked out.\nc. A bacterial protein involved in biomass production of the bacterium and having a structure different from any human proteins.\nd.A bacterial protein involved in biomass production of the bacterium and having a structure very similar to a human protein.\nAns: b, c\n  Consider the MATLAB code:\n[grRatio, grRateKO] = doubleGeneDeletion(model). The MATLAB command sum(grRatio\u0026lt;0.05) gives:\na.The sum of the growth rates that are less than 5% of wild-type growth rate.\nb.The number of reactions for which the growth rate is 5% of wild-type growth rate.\nc.The number of gene-pair deletions for which the growth ratio is less than 0.05.\nd.The number of gene-pair deletions for which the growth rate of the deletion strain is less than 5% of the wild-type growth rate.\nAns: c, d\n  "}),a.add({id:34,href:'/SysBioBook/resources/further-reading/chapter-11/',title:"Chapter 11",section:"Further Reading",content:"Chapter 11: Modelling cellular interactions #  Host–pathogen systems biology\nForst (2006) Drug Discovery Today\nA nice review on host–pathogen systems biology.\n Modelling Viral and Immune System Dynamics\nPerelson (2002) Nature Reviews Immunology\nAn authoritative review on modelling the dynamics of viral infections e.g. HIV/HCV.\n Immunology for physicists\nPerelson (1997) Reviews of Modern Physics\nA detailed review of an extraordinary variety of models of the immune system, from a physicist\u0026rsquo;s perspective.\n Boolean models of within-host immune interactions\nThakar and Albert (2010) Current Opinion in Microbiology\nAn excellent application of Boolean network modelling to study host–pathogen interactions.\n Constraint-based analysis of metabolic capacity of Salmonella typhimurium during host-pathogen interaction\nRaghunathan et al. (2009) BMC Systems Biology\nA nice application of constraint-based modelling to study host–pathogen interactions.\n A systems perspective of host–pathogen interactions: predicting disease outcome in tuberculosis\nRaman et al. (2010) Molecular BioSystems\nA detailed systems-level model of TB infection, capturing a large number of interactions between the pathogen and the host immune system.\n Current trends in modeling host–pathogen interactions\nMukherjee et al. (2013) Wiley Interdisciplinary Reviews: Data Mining and Knowledge Discovery\nAnother nice review on host–pathogen systems biology.\n Metabolic modeling of a mutualistic microbial community\nStolyar et al. (2007) Molecular Systems Biology\nThe classic paper on applying FBA to model microbial communities.\n Enumerating all possible biosynthetic pathways in metabolic networks\nRavikrishnan et al. (2018) Scientific Reports\nDetails the MetQuest approach to enumerate biosynthetic pathways, which can span even across microbes.\n SteadyCom: Predicting microbial abundances while ensuring community stability\nChan et al. (2017) PLoS Computational Biology\nAn excellent paper detailing the background of community modelling approaches, and focussing on the SteadyCom approach, one of the best approaches based on constraint-based modelling.\n Investigating metabolic interactions in a microbial co-culture through integrated modelling and experiments\nRavikrishnan et al. (2020) Computational and Structural Biotechnology Journal\nAn integrated experimental and computational approach, defining an index measuring the metabolic dependency between organisms.\n Modelling approaches for studying the microbiome\nKumar et al. (2019) Nature Microbiology\nA detailed review on metabolic modelling of microbial communities.\n "}),a.add({id:35,href:'/SysBioBook/resources/lecture-videos/chapter-11/',title:"Chapter 11",section:"Lecture Videos",content:"Lecture Videos #  A Massive Open Online Course on Computational Systems Biology, which is in part, a basis for this book, is also available, through the Indian educational platforms, NPTEL (National Programme on Technology Enhanced Learning) and SWAYAM.\nThe lectures (including hands-on lab sessions), totalling over 30 hours of content, are also available on YouTube.\nChapter 11: Modelling cellular interactions #  Computational Modelling of Host\u0026ndash;Pathogen Interactions (15:29)\n Overview of Methods How do we model Mtb-Human Interactions? Reconstructing the Model     Computational Modelling of Host\u0026ndash;Pathogen Interactions (12:36)\n Simulations     "}),a.add({id:36,href:'/SysBioBook/resources/supplementary-exercises/chapter-11/',title:"Chapter 11",section:"Supplementary Exercises",content:"Chapter 11: Modelling cellular interactions #    Which of the following method can be used to identify a set of seed compounds needed for the growth of the organism\na. CarveMe\nb. Reverse Ecology\nc. MetQuest\nd. OptCom\nAns: b\n  Match the following algorithms with their applications\n     Algorithm Application     A.SteadyCom i. Dynamic FBA of the microbial communities   B. COMETS ii. Interaction potential between organism in a community   C. SMETANA iii. Predict community compositions     a. A-i, B-ii, C-iii b. A-iii, B-ii, C-i c. A-iii, B-i, C-ii d. A-ii, B-iii, C-i  Ans: c\n"}),a.add({id:37,href:'/SysBioBook/resources/further-reading/chapter-12/',title:"Chapter 12",section:"Further Reading",content:"Chapter 12: Designing biological circuits #  Introduction to Synthetic Biology\n  A fun over-simplified introduction to synthetic biology, paralleling it with computers.\n Synthetic Biology, Inventing the Future by Andrew Hessel\n  A nice interview of Andrew Hessel, on synthetic biology.\n Synthetic Biology: Honors Colloquium by JJ Collins\n  An excellent talk by JJ Collins, one of the most renowned scientists in synthetic biology.\n Realizing the potential of synthetic biology\nChurch et al. (interview) (2014) Nature Reviews Molecular Cell Biology\nA nice interview-type article on synthetic biology.\n Foundations for engineering biology\nEndy (2005) Nature\nA nice engaging review of synthetic biology.\n Genome-scale engineering for systems and synthetic biology\nEsvelt and Wang (2013) Molecular systems biology\nAn excellent review of technologies and methodologies for genome editing in synthetic biology.\n The second wave of synthetic biology: from modules to systems\nPurnick and Weiss (2009) Nature Reviews Molecular Cell Biology\nA nice overview of multiple synthetic biology circuits.\n The art of molecular computing: Whence and whither\nGangadharan and Raman (2021) BioEssays\nA detailed review of DNA computing, highlighting key experiments across three different eras of molecular computation, beginning with molecular solutions, transitioning to logic circuits and ultimately, more complex molecular networks.\n "}),a.add({id:38,href:'/SysBioBook/resources/lecture-videos/chapter-12/',title:"Chapter 12",section:"Lecture Videos",content:"Lecture Videos #  A Massive Open Online Course on Computational Systems Biology, which is in part, a basis for this book, is also available, through the Indian educational platforms, NPTEL (National Programme on Technology Enhanced Learning) and SWAYAM.\nThe lectures (including hands-on lab sessions), totalling over 30 hours of content, are also available on YouTube.\nChapter 12: Designing biological circuits #  Introduction to Synthetic Biology (32:20)\n Parts Modules Combining Modules Challenges     "}),a.add({id:39,href:'/SysBioBook/resources/supplementary-exercises/chapter-12/',title:"Chapter 12",section:"Supplementary Exercises",content:"Chapter 12: Designing biological circuits #    What are the characteristics of the bow-tie or hour-glass structure?\na. Existence of core central metabolism\nb. Presence of conserved pathways for catabolism and anabolism\nc. Presence of diverse linear pathways for catabolism and anabolism\nd. Existence of diverse central metabolism\nAns: a, c\n  Which of the following strategy can be employed to build a non-retroactive system?\na. Build a system in such a way that the system is accounted for retro-activity\nb. Remove a module from the system if the system alters when that specific module is changed\nc. Inclusion of additional circuit into the system to minimize the interference\nAns: a, c\n  Which of the following can be used as genetic devices to build synthetic biological circuits?\na. Toggle switches\nb. Repressilator\nc. Signalling cascades\nd. All of the above\nAns: d\n  What is the number of topologies possible for a 4-node network, given that every pair of nodes have three different types of interactions, activation, repression and no effect?\n   \\( a. 3^{(4^{3})} \\)   \\( b. 2^{(4^{3})} \\)   \\( c. 3^{(4^{2})} \\)   \\( d. 4^{(2^{3})} \\)  Ans: c\n  Select the examples governing robustness due to the redundancy principle\ni. Synthetic lethals\nii. Gene duplication\niii. Feedback loops\niv. Isozyme expression\nv. Compartmentalization\na. i, ii, iv\nb. ii, iii, iv, v\nc. iii, v\nd. i, ii, iii, iv\nAns: a\n  "}),a.add({id:40,href:'/SysBioBook/resources/further-reading/chapter-13/',title:"Chapter 13",section:"Further Reading",content:"Chapter 13: Robustness and evolvability of biological systems #  Biological Robustness\nKitano (2004) Nature Reviews Genetics\nA classic review of concepts of biological robustness.\n Bow ties, metabolism and disease\nCsete and Doyle (2004) Trends in Biotechnology\nA nice discussion of bow ties in biological systems.\n Complexity and Robustness\nCarlson and Doyle (2002) Proceedings of the National Academy of Sciences USA\nA detailed piece on highly optimised tolerance.\n Evolution of Bow-Tie Architectures in Biology\nFriedlander et al. (2005) PLoS Computational Biology\nA very interesting paper, showing how bow-ties can evolve spontaneously under certain conditions/constraints.\n Neutralism and selectionism: a network-based reconciliation\nWagner (2008) Nature Reviews Genetics\nA wonderful treatise on neutralism vs. selectionism, from a \u0026lsquo;neutral\u0026rsquo; network perspective.\n Innovation and robustness in complex regulatory gene networks\nCiliberti et al. (2007) Proceedings of the National Academy of Sciences USA\nRobustness, evolvability and innovation, as studied in a gene regulatory network system.\n Robustness and evolvability: a paradox resolved\nWagner (2008) Proceedings of the Royal Society B: Biological Sciences\nThe classic paper, beautifully reconciling the seemingly paradoxical relationship between robustness and evolvability.\n Andreas Wagner interview (Zürich Minds)\n  An interview of Andreas Wagner, discussing robustness, evolvability, and innovation.\n "}),a.add({id:41,href:'/SysBioBook/resources/lecture-videos/chapter-13/',title:"Chapter 13",section:"Lecture Videos",content:"Lecture Videos #  A Massive Open Online Course on Computational Systems Biology, which is in part, a basis for this book, is also available, through the Indian educational platforms, NPTEL (National Programme on Technology Enhanced Learning) and SWAYAM.\nThe lectures (including hands-on lab sessions), totalling over 30 hours of content, are also available on YouTube.\nChapter 13: Robustness and evolvability of biological systems #  Robustness in Biological Systems (15:16)\n Overview     Robustness in Biological Systems: Mechanisms (10:47)\n System Control Redundancy Modularity and Decoupling Hierarchies and Protocols     Robustness in Biological Systems: Organising Principles (15:04)\n Bow-Tie structure     Robustness in Biological Systems: Trade-offs (10:12)\n \u0026ldquo;Robustness, Fragility and Complexity\u0026rdquo; Highly Optimised Tolerance Robustness and Evolvability     Robustness and Evolvability (11:29)\n Parametric Robustness `Topological' Robustness Evolvability     Robustness and Evolvability (24:50)\n Neutral Networks Genotype\u0026ndash;Phenotype Mapping     "}),a.add({id:42,href:'/SysBioBook/resources/supplementary-exercises/chapter-13/',title:"Chapter 13",section:"Supplementary Exercises",content:"Chapter 13: Robustness and evolvability of biological systems #  Consider a genotype space of the peptide of length 25. Assume the edges to be the protein evolved by the change in single amino acid.\n  What is the size of the genotype space? (If the answer is   \\( 2.4×10^{41} \\)  , enter your answer as 2e41) Ans: 3e32\n  What is the degree of any genotype in the genotype space? Ans: 475\n  Choose the genotype neighbours of  \\( Ala_{23}-Gly_{2} \\)  in the given genotype space\na.  \\( Glu-Ala_{21}-Gly_{3} \\)  b.  \\( GLu-Ala_{22}-Gly_{2} \\)  c.  \\( Ala_{22}-Ser-Gly-Gly \\)  d.  \\( Ala_{15}-Met-Ala_{7}-Gly_{2} \\)  e.  \\( Ala_{21}-Gly_{4} \\)  Ans: b, c, d\n  Which of the following statements are true about robustness and evolvability?\na. The phenotype robustness and phenotype evolvability are typically positively correlated\nb. The phenotype robustness and phenotype evolvability are typically negatively correlated\nc. The average genotype degree of the neutral network is directly proportional to the phenotype robustness\nd. Robust phenotypes exhibit higher evolvability\nAns: a, c, d\n  Consider two neutral networks A and B having equal number of nodes. Network B has more connected components than Network A, which comprises a single connected component. Which of the following statements is/are true?\na. Phenotype corresponding to Network A likely has higher evolvability\nb. Phenotype corresponding to Network B likely has higher evolvability\nc. Phenotype corresponding to Network A has higher robustness\nd. Phenotype corresponding to Network B has higher robustness\ne. Cannot be concluded with the above information\nAns: a, c\n  "}),a.add({id:43,href:'/SysBioBook/resources/further-reading/chapter-14/',title:"Chapter 14",section:"Further Reading",content:"Chapter 14: Epilogue: The road ahead #  The past 200 years in diabetes\nPolonsky (2012) The New England Journal of Medicine\nAn interesting perspective on two centuries of diabetes.\n Physiological regulation of the pancreatic β-cell: functional insights for understanding and therapy of diabetes\nMcClenaghan (2007) Experimental Physiology\nA detailed overview of pancreatic β-cell regulation, with insights into drug targets and mechanisms.\n Insulin: discovery and controversy\nRosenfeld (2002) Clinical Chemistry\nAn interesting read on insulin.\n The impact of mathematical modeling on the understanding of diabetes and related complications\nAjmera et al. (2013) CPT: Pharmacometrics \u0026amp; Systems Pharmacology\nA solid review on applications of QSP for diabetes modelling.\n Simulating a Living Cell\nCovert (2014) Scientific American\nAn easy-to-read review on whole-cell modelling.\n A Whole-Cell Computational Model Predicts Phenotype from Genotype\nKarr et al. (2012) Cell\nThe classic whole-cell model of Mycoplasma genitalium.\n Bridging the gaps in systems biology\nCvijovic et al. (2014) Molecular Genetics and Genomics\nA nice overview of the field of systems biology.\n "}),a.add({id:44,href:'/SysBioBook/resources/lecture-videos/chapter-14/',title:"Chapter 14",section:"Lecture Videos",content:"Lecture Videos #  A Massive Open Online Course on Computational Systems Biology, which is in part, a basis for this book, is also available, through the Indian educational platforms, NPTEL (National Programme on Technology Enhanced Learning) and SWAYAM.\nThe lectures (including hands-on lab sessions), totalling over 30 hours of content, are also available on YouTube.\nChapter 14: Epilogue: The road ahead #  Guest Lecture: Modelling in Drug Development (18:28)\n Introduction Drug development today: process and challenges     Guest Lecture: Modelling in Drug Development (19:13)\n Modelling in Drug Development     Guest Lecture: Quantitative Systems Pharmacology (29:35)\n Introduction to QSP     Guest Lecture: Quantitative Systems Pharmacology (6:03)\n Case study: QSP model of diabetes     Guest Lecture: Quantitative Systems Pharmacology (12:16)\n What next for QSP Q \u0026amp; A     Advanced Topics (13:46)\n Ensemble Modelling     Advanced Topics (11:31)\n Design Principles     Advanced Topics (27:07)\n Bridging the Gaps in Systems Biology DREAM Challenges Whole-cell Modelling Retroactivity     Course Recap (13:12)\n   "}),a.add({id:45,href:'/SysBioBook/resources/supplementary-exercises/chapter-14/',title:"Chapter 14",section:"Supplementary Exercises",content:"Chapter 14: Epilogue: The road ahead #  "}),a.add({id:46,href:'/SysBioBook/author-info/',title:"About the Author",section:"Home",content:"About the Author #  Karthik Raman is an Associate Professor at the Department of Biotechnology, Bhupat \u0026amp; Jyoti Mehta School of Biosciences, Indian Institute of Technology Madras. He co-founded and co-ordinates the Initiative for Biological Systems Engineering and is a core member of the Robert Bosch Centre for Data Science and Artificial Intelligence (RBCDSAI). He has been a researcher in the area of systems biology for the last 15+ years and has been teaching a course on systems biology for the last eight years, to (mostly) engineers from different backgrounds. His lab works on computational approaches to understand and manipulate biological networks, with applications in metabolic engineering and synthetic biology.\nRamanLab  \nkarthikraman  \nSysBioBook  \nKarthik Raman  \n"}),a.add({id:47,href:'/SysBioBook/errata-feedback/',title:"Errata \u0026 Feedback",section:"Home",content:"Feedback #  Despite my best efforts and those of my colleagues and students who have carefully gone through the book, it is likely that there may be a few typographical/grammatical errors. Do send your feedback via GitHub\u0026mdash;critical feedback will be much appreciated and duly acknowledged!\nErrata #  None so far!\n"}),a.add({id:48,href:'/SysBioBook/faq/',title:"FAQs",section:"Home",content:"Frequently Asked Questions #  What are the pre-requisites for this book?\nThe only pre-requisite for this text is a familiarity with a high-level programming language aside from an intellectual curiosity towards how mathematical and computational approaches aid in understanding biology. An engineering mathematics background will be advantageous but not strictly necessary.  "}),a.add({id:49,href:'/SysBioBook/resources/supplementary-exercises/',title:"Supplementary Exercises",section:"Resources",content:"Supplementary Exercises #    Chapter 1   Introduction to modelling   Chapter 2   Introduction to graph theory   Chapter 3   Structure of networks   Chapter 4   Applications of network biology   Chapter 5   Introduction to dynamic modelling   Chapter 6   Parameter estimation   Chapter 7   Discrete dynamic models: Boolean networks   Chapter 8   Introduction to constraint-based modelling   Chapter 9   Extending constraint-based approaches   Chapter 10   Perturbations to metabolic networks   Chapter 11   Modelling cellular interactions   Chapter 12   Designing biological circuits   Chapter 13   Robustness and evolvability of biological systems   Chapter 14   Epilogue: The road ahead   Appendix A   Introduction to key biological concepts   Appendix B   Reconstruction of biological networks   Appendix C   Databases for systems biology   Appendix D   Software tools compendium   Appendix E   MATLAB for systems biology   "}),a.add({id:50,href:'/SysBioBook/updates/',title:"Updates",section:"Home",content:"Updates #   2021-05-31: The book is officially in print, and available for purchase! 2021-04-28: \u0026ldquo;Introducing Students to Systems Biology\u0026rdquo; blog is up on routledge.com 2021-04-23: Further reading updated, and is effectively complete! 2021-04-05: Lecture videos updated, with topics! 2021-04-05: Further reading updated! 2021-04-05: Details on the appendices are up 2021-03-29: Book cover is up! 2020-10-24: Website is up!  "}),a.add({id:51,href:'/SysBioBook/resources/further-reading/appendix-a/',title:"Appendix A",section:"Further Reading",content:"Chapter A: Introduction to key biological concepts #  "}),a.add({id:52,href:'/SysBioBook/resources/lecture-videos/appendix-a/',title:"Appendix A",section:"Lecture Videos",content:"Lecture Videos #  A Massive Open Online Course on Computational Systems Biology, which is in part, a basis for this book, is also available, through the Indian educational platforms, NPTEL (National Programme on Technology Enhanced Learning) and SWAYAM.\nThe lectures (including hands-on lab sessions), totalling over 30 hours of content, are also available on YouTube.\nChapter A: Introduction to key biological concepts #  "}),a.add({id:53,href:'/SysBioBook/resources/supplementary-exercises/appendix-a/',title:"Appendix A",section:"Supplementary Exercises",content:"Chapter A: Introduction to key biological concepts #  "}),a.add({id:54,href:'/SysBioBook/resources/further-reading/appendix-b/',title:"Appendix B",section:"Further Reading",content:"Chapter B: Reconstruction of biological networks #  Reconstruction of biochemical networks in microorganisms\nFeist et al. (2008) Nature Reviews Microbiology\nA nice review on network reconstruction.\n Deciphering Protein-Protein Interactions, Part I (Experimental Techniques)\nShoemaker and Panchenko (2007) PLoS Computational Biology\nA detailed review on experimental approaches to study protein–protein interactions.\n Deciphering Protein-Protein Interactions, Part II (Computational Techniques)\nShoemaker and Panchenko (2007) PLoS Computational Biology\nA detailed review on computational approaches to study protein–protein interactions.\n Reconstruction of cellular signalling networks and their properties\nPapin et al. (2005) Nature Reviews Molecular Cell Biology\nA nice review on reconstruction of signalling networks.\n A survey of metabolic databases emphasizing the MetaCyc family\nKarp and Caspi (2011) Archives of Toxicology\nA detailed overview of MetaCyc family of metabolic network databases.\n "}),a.add({id:55,href:'/SysBioBook/resources/lecture-videos/appendix-b/',title:"Appendix B",section:"Lecture Videos",content:"Lecture Videos #  A Massive Open Online Course on Computational Systems Biology, which is in part, a basis for this book, is also available, through the Indian educational platforms, NPTEL (National Programme on Technology Enhanced Learning) and SWAYAM.\nThe lectures (including hands-on lab sessions), totalling over 30 hours of content, are also available on YouTube.\nChapter B: Reconstruction of biological networks #  Reconstruction of Gene Regulatory Networks (32:49)\n Reconstruction of Gene Regulatory Networks What is a GRN? Algorithms for reconstruction (overview) Synthetic lethality     Reconstruction of Protein Networks (16:35)\n Rosetta Stone Phylogenetic Profiling Correlated Mutations     Reconstruction of Signalling Networks (12:42)\n Basic Concepts Reconstruction     Reconstruction of Metabolic Networks (14:23)\n Basic Concepts Reconstruction Representations     "}),a.add({id:56,href:'/SysBioBook/resources/supplementary-exercises/appendix-b/',title:"Appendix B",section:"Supplementary Exercises",content:"Chapter B: Reconstruction of biological networks #    The correct order of the key steps in metabolic reconstruction is:\na. Gene functional annotations \u0026gt; model construction \u0026gt; model refinement \u0026gt; model simulation \u0026gt; evaluate model\nb. Gene functional annotations \u0026gt; model refinement \u0026gt; model simulation \u0026gt; evaluate model \u0026gt; model construction\nc. Gene functional annotations \u0026gt; model construction \u0026gt; model simulation \u0026gt; model refinement \u0026gt; evaluate model\nd. Gene functional annotations \u0026gt; model refinement \u0026gt; model construction \u0026gt; model simulation \u0026gt; evaluate model\ne. Gene functional annotations \u0026gt; model construction \u0026gt; model simulation \u0026gt; evaluate model \u0026gt; model refinement\nAns: a\n  Given a stoichiometric matrix   \\( \\mathbf{S} \\)  of dimensions  \\(a \\times b\\)  a. Number of metabolites are  \\(a\\)  b. Number of metabolites are  \\(b\\)  c. Number of reactions are  \\(a\\)  d. If  \\(v\\)  is the vector of fluxes,  \\(\\mathbf{S}\\mathbf{v} = \\mathbf{0}\\)  at steady state\nAns: a, d\n  Which of these can you find in a reconstructed metabolic model of an organism?\na. List of all genes\nb. List of all reactions\nc. GPRs\nd. Stoichiometry of all reactions\ne. Reaction directionality/thermodynamics\nAns: a, b, c, d, e\n  "}),a.add({id:57,href:'/SysBioBook/resources/further-reading/appendix-c/',title:"Appendix C",section:"Further Reading",content:"Chapter C: Databases for systems biology #  "}),a.add({id:58,href:'/SysBioBook/resources/lecture-videos/appendix-c/',title:"Appendix C",section:"Lecture Videos",content:"Lecture Videos #  A Massive Open Online Course on Computational Systems Biology, which is in part, a basis for this book, is also available, through the Indian educational platforms, NPTEL (National Programme on Technology Enhanced Learning) and SWAYAM.\nThe lectures (including hands-on lab sessions), totalling over 30 hours of content, are also available on YouTube.\nChapter C: Databases for systems biology #  "}),a.add({id:59,href:'/SysBioBook/resources/supplementary-exercises/appendix-c/',title:"Appendix C",section:"Supplementary Exercises",content:"Chapter C: Databases for systems biology #  "}),a.add({id:60,href:'/SysBioBook/resources/further-reading/appendix-d/',title:"Appendix D",section:"Further Reading",content:"Chapter D: Software tools compendium #  The Systems Biology Graphical Notation\nLe Novere et al. (2009) Nature Biotechnology\nThe classic paper detailing the SBGN framework.\n "}),a.add({id:61,href:'/SysBioBook/resources/lecture-videos/appendix-d/',title:"Appendix D",section:"Lecture Videos",content:"Lecture Videos #  A Massive Open Online Course on Computational Systems Biology, which is in part, a basis for this book, is also available, through the Indian educational platforms, NPTEL (National Programme on Technology Enhanced Learning) and SWAYAM.\nThe lectures (including hands-on lab sessions), totalling over 30 hours of content, are also available on YouTube.\nChapter D: Software tools compendium #  "}),a.add({id:62,href:'/SysBioBook/resources/supplementary-exercises/appendix-d/',title:"Appendix D",section:"Supplementary Exercises",content:"Chapter D: Software tools compendium #  "}),a.add({id:63,href:'/SysBioBook/resources/further-reading/appendix-e/',title:"Appendix E",section:"Further Reading",content:"Chapter E: MATLAB for systems biology #  Loren: Art of MATLAB\nA very nice blog on the \u0026lsquo;Art\u0026rsquo; of MATLAB, by MATLAB developer Loren Shure.\n What Every Computer Scientist Should Know About Floating Point Arithmetic\nGoldberg (1991) ACM Computing Surveys\nA highly cited classic on what every computer scientist, why everyone, must know about floating-point arithmetic!\n Is Addition on a Computer Associative?\nA nice blog article illustrating some classic pitfalls in floating-point arithmetic.\n Wikipedia: Double-precision Floating-point Format\nWikipedia entry on the floating-point format.\n Get Smart, Be Logical! (2018)\nA nice blog article emphasising the importance of logical indexing in MATLAB.\n "}),a.add({id:64,href:'/SysBioBook/resources/lecture-videos/appendix-e/',title:"Appendix E",section:"Lecture Videos",content:"Lecture Videos #  A Massive Open Online Course on Computational Systems Biology, which is in part, a basis for this book, is also available, through the Indian educational platforms, NPTEL (National Programme on Technology Enhanced Learning) and SWAYAM.\nThe lectures (including hands-on lab sessions), totalling over 30 hours of content, are also available on YouTube.\nChapter E: MATLAB for systems biology #  Lab: MATLAB Basics (12:45)\n Basics of MATLAB     Lab: MATLAB Basics (8:35)\n Indexing     Lab: MATLAB Basics (22:00)\n Matrices \u0026amp; Functions     Lab: MATLAB Basics (13:49)\n Floating Point Arithmetic     "}),a.add({id:65,href:'/SysBioBook/resources/supplementary-exercises/appendix-e/',title:"Appendix E",section:"Supplementary Exercises",content:"Chapter E: MATLAB for systems biology #   What is the ideal way to compare if two floating-point numbers are equal, in MATLAB? a. abs(A-B) \u0026lt; 1e-12\nb. A – B == 0\nc. A == B\nd. floor(A) == floor (B)  Ans: a\n"}),a.add({id:66,href:'/SysBioBook/resources/figure-slides/',title:"Figure Slides",section:"Resources",content:"Figure Slides #  Editable slides in both Powerpoint and PDF (one figure per page) formats are available freely for instructors wishing to use these in their course.\n"})})()